Data	Error thrown
['19.18843', '72.79803', '19.18803', '72.79775', '1501']	<class 'django.db.utils.DatabaseError'>
['19.18498', '72.79765', '', '', '1502']	<class 'django.db.utils.DatabaseError'>
['19.17995', '72.79783', '19.15227', '72.79027', '1503']	<class 'django.db.utils.DatabaseError'>
['19.16781', '72.79065', '19.16790501', '72.79076701', '1506']	<class 'django.db.utils.DatabaseError'>
['19.16335', '72.78989', '19.16322', '72.78948', '1507']	<class 'django.db.utils.DatabaseError'>
['19.15899701', '72.78614601', '19.15858701', '72.78616501', '1509']	<class 'django.db.utils.DatabaseError'>
['19.155801', '72.78788901', '19.15585', '72.78783', '1510']	<class 'django.db.utils.DatabaseError'>
['19.16963801', '72.79589301', '19.1695601', '72.79572401', '2613']	<class 'django.db.utils.DatabaseError'>
['19.15016701', '72.83607201', '', '', '533']	<class 'django.db.utils.DatabaseError'>
['19.11008101', '72.8492901', '19.11061301', '72.84963401', '74']	<class 'django.db.utils.DatabaseError'>
['19.11172', '72.84949', '19.11172', '72.84949', '74']	<class 'django.db.utils.DatabaseError'>
['19.11369', '72.84721', '19.11369', '72.84721', '75']	<class 'django.db.utils.DatabaseError'>
['19.12097', '72.84962', '', '', '76']	<class 'django.db.utils.DatabaseError'>
['19.09669', '72.88357', '19.09658', '72.88368', '362']	<class 'django.db.utils.DatabaseError'>
['19.10708', '72.88295', '19.10753', '72.88068', '364']	<class 'django.db.utils.DatabaseError'>
['19.11531', '72.87979', '', '', '369']	<class 'django.db.utils.DatabaseError'>
['19.10879', '72.85583', '19.10879', '72.85556', '580']	<class 'django.db.utils.DatabaseError'>
['19.10678', '72.85965', '19.10678', '72.85943', '581']	<class 'django.db.utils.DatabaseError'>
['19.10379', '72.86188', '19.10143', '72.86323', '582']	<class 'django.db.utils.DatabaseError'>
['19.10392', '72.86678', '19.10321', '72.86677', '585']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '587']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '588']	<class 'django.db.utils.DatabaseError'>
['19.12483', '72.87269', '', '', '620']	<class 'django.db.utils.DatabaseError'>
['19.1187', '72.85181', '', '', '624']	<class 'django.db.utils.DatabaseError'>
['19.1176', '72.85359', '', '', '625']	<class 'django.db.utils.DatabaseError'>
['19.11389', '72.85924', '19.11548', '72.85744', '627']	<class 'django.db.utils.DatabaseError'>
['19.11389', '72.85931', '19.11498', '72.85813', '627']	<class 'django.db.utils.DatabaseError'>
['19.11726', '72.8625', '19.11723', '72.86287', '628']	<class 'django.db.utils.DatabaseError'>
['19.11783', '72.86804', '', '', '630']	<class 'django.db.utils.DatabaseError'>
['19.11093', '72.85379', '', '', '635']	<class 'django.db.utils.DatabaseError'>
['19.11467', '72.85467', '19.11574', '72.85524', '636']	<class 'django.db.utils.DatabaseError'>
['19.12046', '72.8556', '19.12143', '72.85622', '995']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1163']	<class 'django.db.utils.DatabaseError'>
['19.11314', '72.8642', '19.11333', '72.8639', '1165']	<class 'django.db.utils.DatabaseError'>
['19.11233', '72.8683', '19.11151', '72.86896', '1166']	<class 'django.db.utils.DatabaseError'>
['19.1118', '72.86881', '19.11166', '72.86848', '1166']	<class 'django.db.utils.DatabaseError'>
['19.12217', '72.84787', '', '', '1183']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1184']	<class 'django.db.utils.DatabaseError'>
['19.12474', '72.85142', '', '', '1186']	<class 'django.db.utils.DatabaseError'>
['19.12576', '72.85351', '', '', '1187']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1188']	<class 'django.db.utils.DatabaseError'>
['19.13943', '72.88031', '', '', '1189']	<class 'django.db.utils.DatabaseError'>
['19.12639', '72.85908', '19.12651', '72.85945', '1190']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1193']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1274']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1636']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1686']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1697']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1698']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1700']	<class 'django.db.utils.DatabaseError'>
['19.10678', '72.85965', '', '', '1703']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1709']	<class 'django.db.utils.DatabaseError'>
['19.12508', '72.87282', '', '', '1752']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2117']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2118']	<class 'django.db.utils.DatabaseError'>
['19.11188', '72.85753', '', '', '2154']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2204']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2205']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2206']	<class 'django.db.utils.DatabaseError'>
['19.10392', '72.86678', '', '', '2312']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2313']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2314']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2447']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2474']	<class 'django.db.utils.DatabaseError'>
['19.1142', '72.86175', '19.1142', '72.86175', '2788']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2899']	<class 'django.db.utils.DatabaseError'>
['19.11877', '72.87027', '19.11882', '72.87013', '2936']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2997']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3022']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3113']	<class 'django.db.utils.DatabaseError'>
['19.11037', '72.85132', '19.10997', '72.85184', '3280']	<class 'django.db.utils.DatabaseError'>
['19.11541', '72.84336', '19.11541', '72.84336', '118']	<class 'django.db.utils.DatabaseError'>
['19.11937', '72.84523', '19.11937', '72.84523', '119']	<class 'django.db.utils.DatabaseError'>
['19.12367', '72.84663', '19.12367', '72.84663', '120']	<class 'django.db.utils.DatabaseError'>
['19.12868', '72.84733', '19.12868', '72.84733', '121']	<class 'django.db.utils.DatabaseError'>
['19.13046', '72.82047', '', '', '496']	<class 'django.db.utils.DatabaseError'>
['19.12083', '72.82989', '19.12083', '72.82989', '509']	<class 'django.db.utils.DatabaseError'>
['19.12889', '72.83103', '19.12938', '72.83163', '511']	<class 'django.db.utils.DatabaseError'>
['19.1231', '72.83058', '19.1231', '72.83058', '529']	<class 'django.db.utils.DatabaseError'>
['19.13573', '72.83198', '19.13408', '72.83198', '531']	<class 'django.db.utils.DatabaseError'>
['19.1392', '72.83164', '19.1392', '72.83164', '532']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '618']	<class 'django.db.utils.DatabaseError'>
['19.12873', '72.81639', '', '', '1214']	<class 'django.db.utils.DatabaseError'>
['19.11308', '72.83875', '', '', '1281']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1282']	<class 'django.db.utils.DatabaseError'>
['19.13138', '72.83131', '19.13155', '72.83134', '1363']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1378']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1400']	<class 'django.db.utils.DatabaseError'>
['19.1274', '72.83745', '', '', '1401']	<class 'django.db.utils.DatabaseError'>
['19.12126', '72.83694', '', '', '1402']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1403']	<class 'django.db.utils.DatabaseError'>
['19.12798', '72.84395', '19.12786', '72.84352', '1422']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1423']	<class 'django.db.utils.DatabaseError'>
['19.13038', '72.83751', '', '', '1424']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1425']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1426']	<class 'django.db.utils.DatabaseError'>
['19.13206', '72.84291', '', '', '1448']	<class 'django.db.utils.DatabaseError'>
['19.12988', '72.84244', '', '', '1449']	<class 'django.db.utils.DatabaseError'>
['19.13038', '72.83771', '', '', '1451']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1452']	<class 'django.db.utils.DatabaseError'>
['19.12954', '72.82504', '', '', '2279']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2315']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2381']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2446']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2452']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2454']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2456']	<class 'django.db.utils.DatabaseError'>
['19.12696', '72.81894', '19.12722', '72.81894', '2864']	<class 'django.db.utils.DatabaseError'>
['19.11255', '72.84173', '19.11255', '72.84173', '3348']	<class 'django.db.utils.DatabaseError'>
['19.12926', '72.81745', '', '', '3373']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3586']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3658']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3659']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3972']	<class 'django.db.utils.DatabaseError'>
['19.12654', '72.83939', '19.12654', '72.83939', '3984']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4121']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4122']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4123']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '862']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2111']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2243']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2752']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3337']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3721']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3722']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3723']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3832']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3833']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '220']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '221']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '222']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '223']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '224']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '225']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '226']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1676']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2170']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2171']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2172']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3212']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3213']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3342']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3813']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3814']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '354']	<class 'django.db.utils.DatabaseError'>
['19.04271', '72.93083', '19.04264', '72.93041', '355']	<class 'django.db.utils.DatabaseError'>
['19.04287', '72.92847', '19.04271', '72.92857', '1764']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1821']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1822']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1823']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1824']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2734']	<class 'django.db.utils.DatabaseError'>
['19.04218', '72.93377', '19.04213', '72.93306', '3438']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3493']	<class 'django.db.utils.DatabaseError'>
['19.04218', '72.93377', '', '', '3494']	<class 'django.db.utils.DatabaseError'>
['18.98596', '72.83122', '18.98566', '72.83118', '54']	<class 'django.db.utils.DatabaseError'>
['18.99466', '72.83118', '18.99462', '72.83082', '830']	<class 'django.db.utils.DatabaseError'>
['19.09854', '72.89234', '19.09883', '72.89242', '1694']	<class 'django.db.utils.DatabaseError'>
['19.10159', '72.8902', '19.10185', '72.8902', '1695']	<class 'django.db.utils.DatabaseError'>
['19.10322', '72.88821', '19.10321', '72.8884', '3367']	<class 'django.db.utils.DatabaseError'>
['18.96083', '72.80861', '18.96', '72.80944', '443']	<class 'django.db.utils.DatabaseError'>
['18.90978', '72.81726', '18.90978', '72.81726', '233']	<class 'django.db.utils.DatabaseError'>
['18.91334', '72.82082', '18.9134', '72.8209', '234']	<class 'django.db.utils.DatabaseError'>
['18.91782', '72.8246', '18.91811', '72.82491', '235']	<class 'django.db.utils.DatabaseError'>
['', '', '18.91086', '72.81841', '258']	<class 'django.db.utils.DatabaseError'>
['18.91195', '72.81807', '18.91199', '72.8181', '298']	<class 'django.db.utils.DatabaseError'>
['18.9148', '72.81867', '18.91521', '72.8187', '299']	<class 'django.db.utils.DatabaseError'>
['18.91638', '72.8193', '18.91632', '72.81945', '300']	<class 'django.db.utils.DatabaseError'>
['', '', '18.91627', '72.81747', '308']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '513']	<class 'django.db.utils.DatabaseError'>
['19.0565', '72.84632', '19.05644', '72.84679', '62']	<class 'django.db.utils.DatabaseError'>
['19.06198', '72.8464', '19.06358', '72.84676', '63']	<class 'django.db.utils.DatabaseError'>
['19.06001', '72.84769', '', '', '288']	<class 'django.db.utils.DatabaseError'>
['19.0623', '72.84845', '', '', '290']	<class 'django.db.utils.DatabaseError'>
['19.0577', '72.85326', '19.0577', '72.85326', '837']	<class 'django.db.utils.DatabaseError'>
['19.06031', '72.85788', '19.06031', '72.85788', '839']	<class 'django.db.utils.DatabaseError'>
['19.05619', '72.84762', '19.05612', '72.8476', '955']	<class 'django.db.utils.DatabaseError'>
['19.06366', '72.84778', '', '', '956']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '957']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1150']	<class 'django.db.utils.DatabaseError'>
['19.06096', '72.84952', '', '', '1156']	<class 'django.db.utils.DatabaseError'>
['19.06001', '72.84769', '19.06001', '72.84769', '1356']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1358']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1359']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2037']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2039']	<class 'django.db.utils.DatabaseError'>
['19.06857', '72.86787', '19.06857', '72.86787', '2328']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3470']	<class 'django.db.utils.DatabaseError'>
['19.06605', '72.84795', '', '', '3601']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3615']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3656']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3657']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3660']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3661']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3662']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4000']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4062']	<class 'django.db.utils.DatabaseError'>
['19.06857', '72.86787', '', '', '4063']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4155']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4156']	<class 'django.db.utils.DatabaseError'>
['19.05913', '72.85472', '19.05913', '72.85472', '838']	<class 'django.db.utils.DatabaseError'>
['19.06187', '72.85953', '19.06187', '72.85953', '840']	<class 'django.db.utils.DatabaseError'>
['19.0706', '72.87125', '19.0706', '72.87125', '841']	<class 'django.db.utils.DatabaseError'>
['19.06664', '72.86472', '19.06664', '72.86472', '1616']	<class 'django.db.utils.DatabaseError'>
['', '', '19.06664', '72.86472', '2327']	<class 'django.db.utils.DatabaseError'>
['19.05621', '72.85161', '19.05621', '72.85161', '2470']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3468']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3469']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '45']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '46']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '47']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '48']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '49']	<class 'django.db.utils.DatabaseError'>
['19.05933', '72.83585', '19.05947', '72.83559', '108']	<class 'django.db.utils.DatabaseError'>
['19.05757', '72.83683', '19.05757', '72.83683', '457']	<class 'django.db.utils.DatabaseError'>
['19.06299', '72.83551', '', '', '458']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '919']	<class 'django.db.utils.DatabaseError'>
['19.0535', '72.82406', '19.05278', '72.82419', '921']	<class 'django.db.utils.DatabaseError'>
['19.05486', '72.82957', '19.05495', '72.82946', '922']	<class 'django.db.utils.DatabaseError'>
['19.05527', '72.83118', '', '', '924']	<class 'django.db.utils.DatabaseError'>
['19.0556', '72.83558', '19.05614', '72.83514', '925']	<class 'django.db.utils.DatabaseError'>
['19.0547', '72.82573', '19.05473', '72.82572', '1069']	<class 'django.db.utils.DatabaseError'>
['19.05413', '72.82703', '19.05488', '72.82748', '1072']	<class 'django.db.utils.DatabaseError'>
['19.05896', '72.83215', '19.05952', '72.83093', '1088']	<class 'django.db.utils.DatabaseError'>
['19.05948', '72.83091', '', '', '1089']	<class 'django.db.utils.DatabaseError'>
['19.05976', '72.82613', '19.05978', '72.82616', '1090']	<class 'django.db.utils.DatabaseError'>
['19.05995', '72.82365', '19.05987', '72.82399', '1091']	<class 'django.db.utils.DatabaseError'>
['19.06277', '72.82218', '', '', '1092']	<class 'django.db.utils.DatabaseError'>
['19.06631', '72.82299', '19.06621', '72.82316', '1093']	<class 'django.db.utils.DatabaseError'>
['19.06814', '72.82252', '19.06867', '72.82294', '1094']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1337']	<class 'django.db.utils.DatabaseError'>
['19.04684', '72.81943', '19.04663', '72.81958', '1338']	<class 'django.db.utils.DatabaseError'>
['19.04868', '72.82087', '19.04941', '72.82091', '1339']	<class 'django.db.utils.DatabaseError'>
['19.05077', '72.82136', '19.05181', '72.82199', '1340']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1341']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1343']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1344']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1345']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1346']	<class 'django.db.utils.DatabaseError'>
['', '', '19.05145', '72.82893', '1349']	<class 'django.db.utils.DatabaseError'>
['19.048529', '72.82472', '', '', '1350']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1351']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1352']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1355']	<class 'django.db.utils.DatabaseError'>
['19.05737', '72.83406', '19.05726', '72.83382', '1361']	<class 'django.db.utils.DatabaseError'>
['19.0557', '72.8349', '', '', '2419']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2827']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4001']	<class 'django.db.utils.DatabaseError'>
['19.05332', '72.83795', '19.05311', '72.83805', '']	<type 'exceptions.ValueError'>
['19.05958', '72.82995', '19.05958', '72.82995', '']	<type 'exceptions.ValueError'>
['19.05968', '72.82767', '', '', '']	<type 'exceptions.ValueError'>
['19.05515', '72.8377', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '1954']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1963']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3843']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3844']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3845']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3846']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3847']	<class 'django.db.utils.DatabaseError'>
['', '', '19.14984', '72.94388', '376']	<class 'django.db.utils.DatabaseError'>
['19.1325', '72.93619', '19.13262', '72.93624', '390']	<class 'django.db.utils.DatabaseError'>
['19.1312', '72.93556', '19.13129', '72.93564', '391']	<class 'django.db.utils.DatabaseError'>
['19.13443', '72.93487', '19.13442', '72.93496', '392']	<class 'django.db.utils.DatabaseError'>
['19.13699', '72.93469', '19.13689', '72.93459', '393']	<class 'django.db.utils.DatabaseError'>
['19.14334', '72.93988', '', '', '395']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1912']	<class 'django.db.utils.DatabaseError'>
['19.14821', '72.94813', '', '', '2009']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2063']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2066']	<class 'django.db.utils.DatabaseError'>
['19.14721', '72.94567', '', '', '2290']	<class 'django.db.utils.DatabaseError'>
['19.14899', '72.94409', '', '', '2291']	<class 'django.db.utils.DatabaseError'>
['19.15159', '72.94558', '', '', '2532']	<class 'django.db.utils.DatabaseError'>
['19.14815', '72.94252', '', '', '3446']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3447']	<class 'django.db.utils.DatabaseError'>
['19.14783', '72.94408', '', '', '3448']	<class 'django.db.utils.DatabaseError'>
['19.15271', '72.94628', '', '', '3767']	<class 'django.db.utils.DatabaseError'>
['19.15403', '72.94682', '', '', '3768']	<class 'django.db.utils.DatabaseError'>
['19.15133', '72.94699', '', '', '']	<type 'exceptions.ValueError'>
['19.14969', '72.9489', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '394']	<class 'django.db.utils.DatabaseError'>
['19.13656', '72.92955', '19.13656', '72.92976', '409']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '410']	<class 'django.db.utils.DatabaseError'>
['19.14243', '72.9328', '', '', '411']	<class 'django.db.utils.DatabaseError'>
['19.1468', '72.93657', '', '', '412']	<class 'django.db.utils.DatabaseError'>
['19.15074', '72.93765', '', '', '413']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '414']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '415']	<class 'django.db.utils.DatabaseError'>
['19.14984', '72.94388', '', '', '416']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '418']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '474']	<class 'django.db.utils.DatabaseError'>
['19.15931', '72.93762', '', '', '1609']	<class 'django.db.utils.DatabaseError'>
['19.13981', '72.92834', '', '', '1617']	<class 'django.db.utils.DatabaseError'>
['19.14303', '72.92815', '', '', '1618']	<class 'django.db.utils.DatabaseError'>
['19.14588', '72.92874', '', '', '1619']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1620']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1621']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1622']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1909']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1910']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1911']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1914']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1915']	<class 'django.db.utils.DatabaseError'>
['19.15527', '72.92921', '', '', '2014']	<class 'django.db.utils.DatabaseError'>
['19.14678', '72.9334', '', '', '2015']	<class 'django.db.utils.DatabaseError'>
['19.14744', '72.92943', '', '', '2016']	<class 'django.db.utils.DatabaseError'>
['19.14792', '72.9276', '19.1477', '72.92865', '2017']	<class 'django.db.utils.DatabaseError'>
['19.14933', '72.92724', '', '', '2018']	<class 'django.db.utils.DatabaseError'>
['19.15151', '72.92846', '', '', '2019']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2020']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2021']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2024']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2028']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2029']	<class 'django.db.utils.DatabaseError'>
['19.14001', '72.92559', '19.14', '72.92553', '2030']	<class 'django.db.utils.DatabaseError'>
['19.13908', '72.92369', '', '', '2031']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2034']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2748']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3693']	<class 'django.db.utils.DatabaseError'>
['19.13973', '72.92177', '', '', '3701']	<class 'django.db.utils.DatabaseError'>
['19.1442', '72.93431', '', '', '4098']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2059']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2060']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2061']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2062']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2064']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2065']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2068']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2577']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3598']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3604']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3607']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3608']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3612']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3618']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3623']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3787']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3886']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3887']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3888']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3889']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3890']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3892']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3893']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3894']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4061']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4191']	<class 'django.db.utils.DatabaseError'>
['18.94572', '72.83404', '', '', '10']	<class 'django.db.utils.DatabaseError'>
['18.9451', '72.83416', '18.94678', '72.8354', '10']	<class 'django.db.utils.DatabaseError'>
['18.95428', '72.8318', '18.95884', '72.83176', '12']	<class 'django.db.utils.DatabaseError'>
['18.95213', '72.83402', '18.95213', '72.83402', '12']	<class 'django.db.utils.DatabaseError'>
['18.95953', '72.83176', '18.9595', '72.83172', '14']	<class 'django.db.utils.DatabaseError'>
['18.95921', '72.8316', '18.95921', '72.8316', '14']	<class 'django.db.utils.DatabaseError'>
['18.95962', '72.83155', '18.95962', '72.83155', '14']	<class 'django.db.utils.DatabaseError'>
['18.96319', '72.83201', '', '', '15']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '145']	<class 'django.db.utils.DatabaseError'>
['18.95635', '72.83281', '18.95884', '72.83176', '335']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '691']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '692']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '749']	<class 'django.db.utils.DatabaseError'>
['18.96195', '72.82193', '18.96191', '72.82204', '878']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '879']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3451']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '205']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '301']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '890']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '142']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '144']	<class 'django.db.utils.DatabaseError'>
['19.22045', '72.86546', '19.22062', '72.86592', '659']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '660']	<class 'django.db.utils.DatabaseError'>
['19.23131', '72.86253', '19.22997', '72.86365', '661']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '998']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1387']	<class 'django.db.utils.DatabaseError'>
['19.22112', '72.86065', '', '', '1388']	<class 'django.db.utils.DatabaseError'>
['19.22254', '72.85779', '', '', '1388']	<class 'django.db.utils.DatabaseError'>
['19.22118', '72.86324', '', '', '1576']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1578']	<class 'django.db.utils.DatabaseError'>
['19.23883', '72.86436', '19.23832', '72.86448', '1588']	<class 'django.db.utils.DatabaseError'>
['19.23883', '72.86436', '', '', '1588']	<class 'django.db.utils.DatabaseError'>
['19.24141', '72.86395', '19.24145', '72.86429', '1589']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1593']	<class 'django.db.utils.DatabaseError'>
['19.23311', '72.86317', '', '', '1594']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1599']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1600']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1601']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1604']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1605']	<class 'django.db.utils.DatabaseError'>
['19.24158', '72.86678', '19.24229', '72.86671', '1606']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1608']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2102']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2104']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2105']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2322']	<class 'django.db.utils.DatabaseError'>
['19.24561', '72.87017', '19.24552', '72.87016', '2572']	<class 'django.db.utils.DatabaseError'>
['19.24067', '72.8684', '', '', '2781']	<class 'django.db.utils.DatabaseError'>
['19.22436', '72.8567', '', '', '2878']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2880']	<class 'django.db.utils.DatabaseError'>
['', '', '19.4416', '72.84207', '3417']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3418']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3437']	<class 'django.db.utils.DatabaseError'>
['19.23591', '72.86397', '', '', '3754']	<class 'django.db.utils.DatabaseError'>
['19.22982', '72.86441', '', '', '3763']	<class 'django.db.utils.DatabaseError'>
['19.22895', '72.86489', '', '', '3764']	<class 'django.db.utils.DatabaseError'>
['19.22764', '72.86554', '', '', '3765']	<class 'django.db.utils.DatabaseError'>
['19.2346', '72.86451', '19.2346', '72.86451', '3774']	<class 'django.db.utils.DatabaseError'>
['19.23577', '72.86585', '', '', '3775']	<class 'django.db.utils.DatabaseError'>
['19.23602', '72.86613', '', '', '3776']	<class 'django.db.utils.DatabaseError'>
['19.23605', '72.86693', '', '', '3834']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3835']	<class 'django.db.utils.DatabaseError'>
['19.23657', '72.8689', '', '', '3836']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4133']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4134']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4135']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4136']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4137']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4138']	<class 'django.db.utils.DatabaseError'>
['19.2307', '72.85563', '19.23087', '72.85656', '1234']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1235']	<class 'django.db.utils.DatabaseError'>
['19.23836', '72.85457', '19.23733', '72.85502', '1237']	<class 'django.db.utils.DatabaseError'>
['', '', '19.24072', '72.85446', '1238']	<class 'django.db.utils.DatabaseError'>
['19.22454', '72.85438', '19.22404', '72.85418', '1291']	<class 'django.db.utils.DatabaseError'>
['19.22616', '72.85493', '19.22637', '72.85515', '1292']	<class 'django.db.utils.DatabaseError'>
['19.23198', '72.85613', '19.23198', '72.85613', '1293']	<class 'django.db.utils.DatabaseError'>
['19.2344', '72.85488', '19.23442', '72.85519', '1294']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1306']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1310']	<class 'django.db.utils.DatabaseError'>
['19.24254', '72.85205', '19.2422', '72.85233', '1311']	<class 'django.db.utils.DatabaseError'>
['19.24429', '72.85177', '', '', '1312']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1313']	<class 'django.db.utils.DatabaseError'>
['19.24234', '72.8468', '', '', '1362']	<class 'django.db.utils.DatabaseError'>
['19.23107', '72.84061', '', '', '1368']	<class 'django.db.utils.DatabaseError'>
['19.22866', '72.8444', '', '', '1369']	<class 'django.db.utils.DatabaseError'>
['19.22861', '72.84846', '19.22868', '72.84846', '1370']	<class 'django.db.utils.DatabaseError'>
['19.22895', '72.85147', '19.22876', '72.85132', '1371']	<class 'django.db.utils.DatabaseError'>
['19.24418', '72.85024', '19.24403', '72.84997', '1373']	<class 'django.db.utils.DatabaseError'>
['19.24394', '72.8486', '19.24289', '72.85397', '1374']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1412']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1413']	<class 'django.db.utils.DatabaseError'>
['19.23493', '72.84593', '19.2348', '72.84604', '1414']	<class 'django.db.utils.DatabaseError'>
['19.23196', '72.84675', '19.23233', '72.84676', '1415']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1417']	<class 'django.db.utils.DatabaseError'>
['', '', '19.1325', '72.513', '1438']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1439']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1441']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1442']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1443']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1444']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1584']	<class 'django.db.utils.DatabaseError'>
['19.23502', '72.84219', '19.23459', '72.84202', '1586']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2190']	<class 'django.db.utils.DatabaseError'>
['19.23433', '72.84071', '', '', '2224']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2603']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2604']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2653']	<class 'django.db.utils.DatabaseError'>
['19.23055', '72.84807', '', '', '2726']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2745']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2762']	<class 'django.db.utils.DatabaseError'>
['19.23075', '72.85062', '19.23096', '72.8509', '2855']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2858']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2859']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2860']	<class 'django.db.utils.DatabaseError'>
['19.23517', '72.8437', '19.23512', '72.84432', '2888']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2926']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2985']	<class 'django.db.utils.DatabaseError'>
['19.23381', '72.85811', '', '', '3361']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3412']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3413']	<class 'django.db.utils.DatabaseError'>
['19.23332', '72.84128', '', '', '3690']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3744']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3745']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3746']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3747']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3748']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3749']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3750']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3830']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3831']	<class 'django.db.utils.DatabaseError'>
['19.23239', '72.84434', '', '', '3953']	<class 'django.db.utils.DatabaseError'>
['19.23305', '72.84282', '', '', '3954']	<class 'django.db.utils.DatabaseError'>
['19.23055', '72.84807', '19.2307', '72.84807', '4132']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '']	<type 'exceptions.ValueError'>
['18.97775', '72.83378', '18.97775', '72.83378', '18']	<class 'django.db.utils.DatabaseError'>
['18.98125', '72.83336', '18.98125', '72.83478', '19']	<class 'django.db.utils.DatabaseError'>
['18.98102', '72.83336', '18.98102', '72.8346', '19']	<class 'django.db.utils.DatabaseError'>
['18.98475', '72.83521', '18.98594', '72.83578', '20']	<class 'django.db.utils.DatabaseError'>
['18.98559', '72.83527', '', '', '20']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '104']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '105']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '106']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '773']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1139']	<class 'django.db.utils.DatabaseError'>
['18.97376', '72.8344', '', '', '1140']	<class 'django.db.utils.DatabaseError'>
['18.97603', '72.83196', '18.9771', '72.83207', '77']	<class 'django.db.utils.DatabaseError'>
['18.98075', '72.83188', '18.98006', '72.83215', '829']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1145']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1146']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '873']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '874']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '891']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1105']	<class 'django.db.utils.DatabaseError'>
['19.11295', '72.88831', '19.11295', '72.88831', '370']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1681']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1682']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1683']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1919']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1921']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2310']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2457']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3713']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3714']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3715']	<class 'django.db.utils.DatabaseError'>
['19.11166', '72.88826', '19.11099', '72.88804', '3839']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3896']	<class 'django.db.utils.DatabaseError'>
['', '', '19.642', '72.543', '3985']	<class 'django.db.utils.DatabaseError'>
['19.21995', '72.82868', '', '', '959']	<class 'django.db.utils.DatabaseError'>
['19.20792', '72.82942', '19.20799', '72.82941', '975']	<class 'django.db.utils.DatabaseError'>
['19.21037', '72.82887', '19.21032', '72.82883', '976']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '977']	<class 'django.db.utils.DatabaseError'>
['19.21538', '72.8291', '', '', '978']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '979']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1319']	<class 'django.db.utils.DatabaseError'>
['19.20954', '72.82259', '19.20931', '72.82261', '1320']	<class 'django.db.utils.DatabaseError'>
['19.21546', '72.82605', '', '', '1432']	<class 'django.db.utils.DatabaseError'>
['19.21463', '72.82085', '19.21476', '72.82089', '1434']	<class 'django.db.utils.DatabaseError'>
['19.21217', '72.82088', '19.2119', '72.82074', '1435']	<class 'django.db.utils.DatabaseError'>
['19.20897', '72.82112', '', '', '1436']	<class 'django.db.utils.DatabaseError'>
['19.21042', '72.82663', '', '', '1437']	<class 'django.db.utils.DatabaseError'>
['19.21051', '72.82435', '19.21048', '72.82433', '1445']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1447']	<class 'django.db.utils.DatabaseError'>
['19.21663', '72.83397', '19.21631', '72.83395', '2126']	<class 'django.db.utils.DatabaseError'>
['19.2193', '72.83346', '19.21913', '72.83358', '2127']	<class 'django.db.utils.DatabaseError'>
['19.21552', '72.82234', '', '', '2830']	<class 'django.db.utils.DatabaseError'>
['19.21536', '72.83107', '', '', '2872']	<class 'django.db.utils.DatabaseError'>
['19.2155', '72.8246', '', '', '3694']	<class 'django.db.utils.DatabaseError'>
['19.21663', '72.83397', '', '', '3718']	<class 'django.db.utils.DatabaseError'>
['19.214872', '72.8131', '19.21486', '72.81307', '3719']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3720']	<class 'django.db.utils.DatabaseError'>
['19.21507', '72.81613', '', '', '4157']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4158']	<class 'django.db.utils.DatabaseError'>
['19.21548', '72.81988', '19.21565', '72.81976', '4159']	<class 'django.db.utils.DatabaseError'>
['18.94898', '72.82154', '18.94909', '72.82166', '440']	<class 'django.db.utils.DatabaseError'>
['18.95101', '72.82334', '18.95102', '72.82333', '682']	<class 'django.db.utils.DatabaseError'>
['18.95433', '72.81979', '18.95378', '72.82042', '683']	<class 'django.db.utils.DatabaseError'>
['18.95111', '72.81972', '18.95093', '72.821', '3145']	<class 'django.db.utils.DatabaseError'>
['19.05594', '72.88817', '', '', '174']	<class 'django.db.utils.DatabaseError'>
['19.05269', '72.8906', '', '', '175']	<class 'django.db.utils.DatabaseError'>
['19.05568', '72.89493', '19.05516', '72.89376', '176']	<class 'django.db.utils.DatabaseError'>
['19.07775', '72.90388', '', '', '198']	<class 'django.db.utils.DatabaseError'>
['19.05152', '72.90054', '', '', '199']	<class 'django.db.utils.DatabaseError'>
['19.04585', '72.90421', '', '', '348']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '810']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '811']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '812']	<class 'django.db.utils.DatabaseError'>
['19.05179', '72.91048', '19.0524', '72.91083', '1830']	<class 'django.db.utils.DatabaseError'>
['19.04714', '72.90953', '19.0471', '72.90972', '1831']	<class 'django.db.utils.DatabaseError'>
['19.05374', '72.88472', '', '', '173']	<class 'django.db.utils.DatabaseError'>
['19.04804', '72.89335', '', '', '177']	<class 'django.db.utils.DatabaseError'>
['19.04552', '72.89407', '', '', '178']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '179']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '180']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '181']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '182']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '186']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '187']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '188']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '189']	<class 'django.db.utils.DatabaseError'>
['19.05415', '72.89336', '', '', '197']	<class 'django.db.utils.DatabaseError'>
['19.05898', '72.9005', '19.059', '72.90065', '201']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '201']	<class 'django.db.utils.DatabaseError'>
['19.06035', '72.90467', '19.06019', '72.90479', '339']	<class 'django.db.utils.DatabaseError'>
['19.059', '72.90752', '19.05857', '72.90791', '340']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '346']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '347']	<class 'django.db.utils.DatabaseError'>
['19.0542', '72.89186', '', '', '809']	<class 'django.db.utils.DatabaseError'>
['19.05769', '72.89545', '19.05769', '72.89545', '1081']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1082']	<class 'django.db.utils.DatabaseError'>
['19.06098', '72.90908', '19.06111', '72.90856', '1757']	<class 'django.db.utils.DatabaseError'>
['19.06195', '72.90603', '19.06217', '72.90572', '1758']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1765']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1766']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1804']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1809']	<class 'django.db.utils.DatabaseError'>
['19.06132', '72.8975', '19.06086', '72.89742', '1829']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2823']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3339']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3460']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3461']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '184']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '185']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '345']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1084']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1260']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1759']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1785']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1786']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1787']	<class 'django.db.utils.DatabaseError'>
['', '', '19.05613', '72.91004', '1788']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1799']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1808']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2425']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2903']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2905']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2939']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3726']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3733']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3756']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3757']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3758']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3759']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3902']	<class 'django.db.utils.DatabaseError'>
['18.94088', '72.83452', '18.9417', '72.83477', '9']	<class 'django.db.utils.DatabaseError'>
['18.94096', '72.83462', '18.94131', '72.83477', '9']	<class 'django.db.utils.DatabaseError'>
['18.942', '72.83468', '18.94099', '72.8344', '9']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '156']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '160']	<class 'django.db.utils.DatabaseError'>
['19.05083', '72.87454', '19.05064', '72.87493', '170']	<class 'django.db.utils.DatabaseError'>
['19.05248', '72.87911', '', '', '171']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1054']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1066']	<class 'django.db.utils.DatabaseError'>
['18.93303', '72.82669', '18.93306', '72.82668', '131']	<class 'django.db.utils.DatabaseError'>
['18.93595', '72.82401', '18.93552', '72.82413', '497']	<class 'django.db.utils.DatabaseError'>
['18.93861', '72.82417', '', '', '1112']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3773']	<class 'django.db.utils.DatabaseError'>
['18.91401', '72.82275', '18.91225', '72.82201', '1']	<class 'django.db.utils.DatabaseError'>
['18.9155', '72.82559', '18.91587', '72.82619', '2']	<class 'django.db.utils.DatabaseError'>
['18.91698', '72.82709', '18.91736', '72.82736', '3']	<class 'django.db.utils.DatabaseError'>
['18.92017', '72.83016', '18.92086', '72.83053', '4']	<class 'django.db.utils.DatabaseError'>
['18.92222', '72.83111', '', '', '5']	<class 'django.db.utils.DatabaseError'>
['18.90023', '72.81373', '18.90023', '72.81373', '37']	<class 'django.db.utils.DatabaseError'>
['18.90476', '72.81575', '18.9047', '72.81579', '39']	<class 'django.db.utils.DatabaseError'>
['18.90719', '72.81702', '18.90704', '72.81682', '40']	<class 'django.db.utils.DatabaseError'>
['18.90758', '72.80894', '18.90763', '72.80888', '87']	<class 'django.db.utils.DatabaseError'>
['18.90706', '72.81228', '18.90689', '72.81284', '89']	<class 'django.db.utils.DatabaseError'>
['18.91533', '72.82419', '18.91533', '72.82419', '3453']	<class 'django.db.utils.DatabaseError'>
['18.90259', '72.81507', '', '', '3454']	<class 'django.db.utils.DatabaseError'>
['18.91117', '72.82094', '18.91114', '72.82091', '3589']	<class 'django.db.utils.DatabaseError'>
['18.92417', '72.83188', '18.92417', '72.83188', '3676']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3699']	<class 'django.db.utils.DatabaseError'>
['18.9076', '72.81199', '18.9076', '72.81199', '3702']	<class 'django.db.utils.DatabaseError'>
['18.91017', '72.81322', '', '', '3703']	<class 'django.db.utils.DatabaseError'>
['18.92086', '72.83053', '18.92086', '72.83053', '3704']	<class 'django.db.utils.DatabaseError'>
['18.92138', '72.82724', '18.92138', '72.82724', '']	<type 'exceptions.ValueError'>
['18.92299', '72.83178', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '763']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '764']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1657']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3371']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '23']	<class 'django.db.utils.DatabaseError'>
['19.01164', '72.84328', '19.01154', '72.84334', '27']	<class 'django.db.utils.DatabaseError'>
['19.0122', '72.84371', '19.0122', '72.84371', '27']	<class 'django.db.utils.DatabaseError'>
['19.01848', '72.84823', '', '', '28']	<class 'django.db.utils.DatabaseError'>
['19.01848', '72.84804', '', '', '29']	<class 'django.db.utils.DatabaseError'>
['19.01896', '72.84833', '', '', '29']	<class 'django.db.utils.DatabaseError'>
['19.01949', '72.84884', '', '', '29']	<class 'django.db.utils.DatabaseError'>
['19.01973', '72.84892', '', '', '29']	<class 'django.db.utils.DatabaseError'>
['19.01715', '72.84717', '', '', '29']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '336']	<class 'django.db.utils.DatabaseError'>
['19.01406', '72.84643', '', '', '662']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '938']	<class 'django.db.utils.DatabaseError'>
['19.02154', '72.84743', '', '', '939']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '940']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '941']	<class 'django.db.utils.DatabaseError'>
['19.02095', '72.84218', '19.02095', '72.84218', '30']	<class 'django.db.utils.DatabaseError'>
['19.01976', '72.83745', '19.01998', '72.83641', '83']	<class 'django.db.utils.DatabaseError'>
['19.01805', '72.83548', '19.01805', '72.83548', '83']	<class 'django.db.utils.DatabaseError'>
['19.02224', '72.83782', '19.02224', '72.83782', '405']	<class 'django.db.utils.DatabaseError'>
['19.01514', '72.83703', '19.0163', '72.83767', '549']	<class 'django.db.utils.DatabaseError'>
['19.01866', '72.84002', '19.01842', '72.83972', '550']	<class 'django.db.utils.DatabaseError'>
['19.0137', '72.83758', '', '', '907']	<class 'django.db.utils.DatabaseError'>
['19.02109', '72.84378', '19.02109', '72.84378', '909']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '910']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1143']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4153']	<class 'django.db.utils.DatabaseError'>
['19.01683', '72.84055', '19.01665', '72.84052', '']	<type 'exceptions.ValueError'>
['', '', '', '', '1002']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1004']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1006']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1007']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1298']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1315']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1329']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1330']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1331']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1334']	<class 'django.db.utils.DatabaseError'>
['19.24398', '72.86518', '', '', '1590']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1592']	<class 'django.db.utils.DatabaseError'>
['19.24425', '72.86721', '19.24421', '72.86728', '1607']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1951']	<class 'django.db.utils.DatabaseError'>
['19.25972', '72.86194', '', '', '1952']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2323']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2324']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2325']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2326']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2353']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2354']	<class 'django.db.utils.DatabaseError'>
['19.2363', '72.86415', '19.2363', '72.86415', '2355']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2706']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2707']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2708']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2709']	<class 'django.db.utils.DatabaseError'>
['19.24039', '72.86983', '', '', '2780']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3172']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3173']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3175']	<class 'django.db.utils.DatabaseError'>
['19.25472', '72.8625', '', '', '3201']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3202']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3270']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3303']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3362']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3671']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3992']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3993']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4102']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4103']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1223']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1239']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1241']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1242']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1295']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1296']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1303']	<class 'django.db.utils.DatabaseError'>
['19.24972', '72.85889', '', '', '1304']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1305']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1411']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1418']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2741']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2762']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2763']	<class 'django.db.utils.DatabaseError'>
['19.2475', '72.85889', '', '', '3360']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3751']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3752']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3753']	<class 'django.db.utils.DatabaseError'>
['18.99157', '72.83149', '18.99093', '72.83172', '55']	<class 'django.db.utils.DatabaseError'>
['18.98577', '72.83176', '18.98577', '72.83176', '']	<type 'exceptions.ValueError'>
['19.04572', '72.90766', '', '', '349']	<class 'django.db.utils.DatabaseError'>
['19.04551', '72.91119', '', '', '350']	<class 'django.db.utils.DatabaseError'>
['19.04562', '72.9146', '', '', '351']	<class 'django.db.utils.DatabaseError'>
['19.04497', '72.91765', '', '', '352']	<class 'django.db.utils.DatabaseError'>
['19.04391', '72.92127', '', '', '353']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1087']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1791']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1792']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1793']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2989']	<class 'django.db.utils.DatabaseError'>
['19.05836', '72.91777', '', '', '3376']	<class 'django.db.utils.DatabaseError'>
['19.05018', '72.91995', '', '', '3769']	<class 'django.db.utils.DatabaseError'>
['19.05223', '72.91695', '', '', '3770']	<class 'django.db.utils.DatabaseError'>
['19.04678', '72.91975', '', '', '3771']	<class 'django.db.utils.DatabaseError'>
['19.04535', '72.92523', '', '', '']	<type 'exceptions.ValueError'>
['19.05148', '72.85836', '19.05111', '72.85811', '286']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '928']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '932']	<class 'django.db.utils.DatabaseError'>
['19.0485', '72.85548', '19.04834', '72.85523', '950']	<class 'django.db.utils.DatabaseError'>
['19.05161', '72.86173', '', '', '952']	<class 'django.db.utils.DatabaseError'>
['19.04753', '72.85245', '', '', '954']	<class 'django.db.utils.DatabaseError'>
['19.04809', '72.85915', '', '', '958']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1153']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1155']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2794']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3346']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3372']	<class 'django.db.utils.DatabaseError'>
['19.0541', '72.89167', '', '', '3731']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3777']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3778']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3779']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '143']	<class 'django.db.utils.DatabaseError'>
['18.94608', '72.8317', '18.94607', '72.8317', '202']	<class 'django.db.utils.DatabaseError'>
['18.94463', '72.83034', '18.94456', '72.83024', '815']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1134']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4143']	<class 'django.db.utils.DatabaseError'>
['19.17551', '72.86472', '19.17526', '72.86509', '574']	<class 'django.db.utils.DatabaseError'>
['19.17469', '72.86053', '19.17577', '72.85946', '647']	<class 'django.db.utils.DatabaseError'>
['19.1715', '72.87724', '', '', '1472']	<class 'django.db.utils.DatabaseError'>
['19.17591', '72.86713', '19.17557', '72.86701', '1489']	<class 'django.db.utils.DatabaseError'>
['19.17569', '72.8733', '', '', '1491']	<class 'django.db.utils.DatabaseError'>
['19.17289', '72.87544', '19.16896', '72.87552', '1492']	<class 'django.db.utils.DatabaseError'>
['19.16566', '72.8782', '19.16547', '72.87819', '1723']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1724']	<class 'django.db.utils.DatabaseError'>
['19.17223', '72.86988', '19.17223', '72.8703', '1736']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1738']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1747']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1748']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1749']	<class 'django.db.utils.DatabaseError'>
['19.17389', '72.87737', '', '', '2803']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2892']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2893']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2929']	<class 'django.db.utils.DatabaseError'>
['19.17329', '72.87951', '', '', '2946']	<class 'django.db.utils.DatabaseError'>
['19.16761', '72.87669', '19.16731', '72.87666', '3005']	<class 'django.db.utils.DatabaseError'>
['19.17621', '72.87621', '', '', '3421']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3422']	<class 'django.db.utils.DatabaseError'>
['19.17599', '72.87766', '', '', '3423']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3424']	<class 'django.db.utils.DatabaseError'>
['19.16769', '72.88138', '19.16766', '72.88129', '3425']	<class 'django.db.utils.DatabaseError'>
['19.16999', '72.88143', '19.16991', '72.88132', '3435']	<class 'django.db.utils.DatabaseError'>
['19.1715', '72.87724', '', '', '3457']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3458']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3459']	<class 'django.db.utils.DatabaseError'>
['19.17242', '72.88082', '', '', '3572']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3574']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4042']	<class 'django.db.utils.DatabaseError'>
['18.97152', '72.84518', '18.97192', '72.84512', '271']	<class 'django.db.utils.DatabaseError'>
['18.95904', '72.84612', '18.95908', '72.84614', '668']	<class 'django.db.utils.DatabaseError'>
['18.96413', '72.84655', '18.96394', '72.84651', '669']	<class 'django.db.utils.DatabaseError'>
['18.96836', '72.84374', '18.96848', '72.84353', '671']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '672']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '673']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '684']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '720']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '685']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '686']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '689']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '690']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '693']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '719']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '57']	<class 'django.db.utils.DatabaseError'>
['19.0104', '72.83125', '19.01164', '72.83244', '78']	<class 'django.db.utils.DatabaseError'>
['19.00851', '72.83064', '', '', '191']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '780']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '904']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '905']	<class 'django.db.utils.DatabaseError'>
['19.00601', '72.8389', '', '', '906']	<class 'django.db.utils.DatabaseError'>
['19.01159', '72.83243', '19.01159', '72.83243', '3760']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3761']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3762']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '374']	<class 'django.db.utils.DatabaseError'>
['18.93391', '72.83217', '18.93411', '72.83216', '7']	<class 'django.db.utils.DatabaseError'>
['18.94381', '72.83924', '18.94381', '72.83924', '7']	<class 'django.db.utils.DatabaseError'>
['18.94381', '72.83924', '18.94381', '72.83924', '7']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '8']	<class 'django.db.utils.DatabaseError'>
['18.93035', '72.83524', '18.92691', '72.83372', '91']	<class 'django.db.utils.DatabaseError'>
['18.93035', '72.83524', '18.93005', '72.83481', '92']	<class 'django.db.utils.DatabaseError'>
['18.93405', '72.83972', '18.93413', '72.83985', '93']	<class 'django.db.utils.DatabaseError'>
['18.9385', '72.83661', '18.93843', '72.83654', '94']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '140']	<class 'django.db.utils.DatabaseError'>
['18.93284', '72.83654', '18.93263', '72.83621', '705']	<class 'django.db.utils.DatabaseError'>
['18.92582', '72.83328', '', '', '706']	<class 'django.db.utils.DatabaseError'>
['18.94727', '72.84012', '', '', '707']	<class 'django.db.utils.DatabaseError'>
['18.93707', '72.83689', '18.93685', '72.83681', '2120']	<class 'django.db.utils.DatabaseError'>
['18.93685', '72.83681', '18.93405', '72.83972', '2120']	<class 'django.db.utils.DatabaseError'>
['18.94434', '72.8393', '18.94434', '72.8393', '3452']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3670']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3674']	<class 'django.db.utils.DatabaseError'>
['19.13195', '72.82551', '', '', '616']	<class 'django.db.utils.DatabaseError'>
['19.13357', '72.82578', '19.1338', '72.82561', '617']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2453']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2455']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1894']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3960']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3961']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3962']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3963']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3964']	<class 'django.db.utils.DatabaseError'>
['19.06675', '72.89967', '19.06614', '72.90068', '321']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '322']	<class 'django.db.utils.DatabaseError'>
['19.081339', '72.9045305555556', '', '', '323']	<class 'django.db.utils.DatabaseError'>
['19.08351', '72.91394', '', '', '324']	<class 'django.db.utils.DatabaseError'>
['19.08582', '72.91464', '', '', '325']	<class 'django.db.utils.DatabaseError'>
['19.08818', '72.91699', '', '', '326']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '333']	<class 'django.db.utils.DatabaseError'>
['19.07373', '72.91168', '', '', '378']	<class 'django.db.utils.DatabaseError'>
['19.078', '72.91595', '', '', '379']	<class 'django.db.utils.DatabaseError'>
['19.0748', '72.91299', '19.0748', '72.91299', '380']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1770']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1771']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1772']	<class 'django.db.utils.DatabaseError'>
['19.07485', '72.90314', '', '', '1773']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1774']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1775']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1777']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1832']	<class 'django.db.utils.DatabaseError'>
['19.0795', '72.90401', '', '', '1833']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1834']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1835']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1839']	<class 'django.db.utils.DatabaseError'>
['19.07438', '72.90662', '', '', '1906']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1907']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2225']	<class 'django.db.utils.DatabaseError'>
['19.07713', '72.90957', '', '', '2226']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2633']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2914']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2921']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2922']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3736']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3737']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3742']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3743']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3819']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3820']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4126']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '246']	<class 'django.db.utils.DatabaseError'>
['19.08659', '72.89725', '19.08652', '72.89799', '248']	<class 'django.db.utils.DatabaseError'>
['19.08715', '72.90001', '19.08686', '72.89973', '249']	<class 'django.db.utils.DatabaseError'>
['19.08768', '72.90322', '19.08758', '72.90282', '250']	<class 'django.db.utils.DatabaseError'>
['19.08876', '72.90681', '19.08875', '72.90678', '251']	<class 'django.db.utils.DatabaseError'>
['19.0907', '72.90984', '', '', '253']	<class 'django.db.utils.DatabaseError'>
['19.093', '72.91241', '', '', '254']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '259']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '260']	<class 'django.db.utils.DatabaseError'>
['19.0895', '72.90269', '', '', '262']	<class 'django.db.utils.DatabaseError'>
['19.09114', '72.90237', '', '', '263']	<class 'django.db.utils.DatabaseError'>
['19.09395', '72.89894', '', '', '264']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '265']	<class 'django.db.utils.DatabaseError'>
['19.08625', '72.90856', '', '', '282']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '358']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1381']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1696']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1850']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1851']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1852']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1854']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1916']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1917']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1918']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2011']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2012']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2013']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2473']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2612']	<class 'django.db.utils.DatabaseError'>
['19.08738', '72.90586', '', '', '3306']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3672']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4072']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3782']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3783']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3784']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3785']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3786']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3787']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3788']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3789']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3790']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3791']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3792']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3793']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3794']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3795']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3796']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3797']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3798']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3799']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3800']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3801']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3802']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3821']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3822']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3823']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3824']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3969']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3970']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3971']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3973']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3978']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3979']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4002']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4003']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4004']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4005']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4006']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4109']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4184']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4185']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4186']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4188']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4189']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '']	<type 'exceptions.ValueError'>
['18.9502', '72.82534', '18.94948', '72.82479', '679']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '680']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '683']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '872']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3145']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3165']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1364']	<class 'django.db.utils.DatabaseError'>
['19.23211', '72.83043', '19.23209', '72.83142', '1365']	<class 'django.db.utils.DatabaseError'>
['19.23181', '72.83286', '19.23177', '72.83337', '1366']	<class 'django.db.utils.DatabaseError'>
['19.23047', '72.83851', '19.23062', '72.83806', '1367']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1453']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1596']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2122']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2123']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2856']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2857']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3414']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3415']	<class 'django.db.utils.DatabaseError'>
['19.15051', '72.8566', '', '', '642']	<class 'django.db.utils.DatabaseError'>
['19.15371', '72.85676', '19.15371', '72.85678', '643']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '644']	<class 'django.db.utils.DatabaseError'>
['', '', '19.17065', '72.85923', '646']	<class 'django.db.utils.DatabaseError'>
['19.13654', '72.88174', '19.13648', '72.88183', '1270']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1271']	<class 'django.db.utils.DatabaseError'>
['19.15797', '72.87442', '19.15788', '72.87446', '1272']	<class 'django.db.utils.DatabaseError'>
['19.16484', '72.86749', '19.16485', '72.86725', '1273']	<class 'django.db.utils.DatabaseError'>
['19.17747', '72.83707', '', '', '1480']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1481']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1484']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1485']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1487']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1488']	<class 'django.db.utils.DatabaseError'>
['19.14827', '72.87658', '19.14857', '72.87657', '1720']	<class 'django.db.utils.DatabaseError'>
['19.15429', '72.87876', '19.15416', '72.87893', '1721']	<class 'django.db.utils.DatabaseError'>
['19.15833', '72.87907', '19.15913', '72.88222', '1722']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1725']	<class 'django.db.utils.DatabaseError'>
['19.14483', '72.87764', '19.14459', '72.87778', '1726']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1727']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1729']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1730']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1731']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1732']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1733']	<class 'django.db.utils.DatabaseError'>
['19.17377', '72.86824', '', '', '1745']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1746']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1947']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2618']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2666']	<class 'django.db.utils.DatabaseError'>
['19.15581', '72.88422', '19.15581', '72.88422', '2729']	<class 'django.db.utils.DatabaseError'>
['19.15713', '72.88288', '', '', '2730']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3195']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3196']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3198']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3283']	<class 'django.db.utils.DatabaseError'>
['19.15273', '72.88212', '19.15093', '72.88212', '3488']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3489']	<class 'django.db.utils.DatabaseError'>
['', '', '19.15272', '72.88046', '3490']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3667']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3668']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3675']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3679']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3682']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3684']	<class 'django.db.utils.DatabaseError'>
['', '', '19.1013', '72.5219', '3685']	<class 'django.db.utils.DatabaseError'>
['19.17198', '72.87115', '19.17198', '72.87127', '3689']	<class 'django.db.utils.DatabaseError'>
['19.16414', '72.87026', '19.16438', '72.87002', '3697']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3734']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3988']	<class 'django.db.utils.DatabaseError'>
['19.17195', '72.87239', '19.17195', '72.87219', '4041']	<class 'django.db.utils.DatabaseError'>
['19.15395', '72.87523', '19.15443', '72.87507', '']	<type 'exceptions.ValueError'>
['19.1464', '72.84298', '19.14655', '72.84293', '124']	<class 'django.db.utils.DatabaseError'>
['19.15715', '72.83504', '', '', '126']	<class 'django.db.utils.DatabaseError'>
['19.17196', '72.84522', '', '', '294']	<class 'django.db.utils.DatabaseError'>
['19.15276', '72.83552', '', '', '522']	<class 'django.db.utils.DatabaseError'>
['19.158532', '72.84361', '', '', '538']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '540']	<class 'django.db.utils.DatabaseError'>
['19.15174', '72.84424', '19.15174', '72.84424', '557']	<class 'django.db.utils.DatabaseError'>
['19.15389', '72.84505', '', '', '558']	<class 'django.db.utils.DatabaseError'>
['19.15703', '72.84578', '', '', '559']	<class 'django.db.utils.DatabaseError'>
['19.16211', '72.84604', '', '', '560']	<class 'django.db.utils.DatabaseError'>
['19.16404', '72.84585', '', '', '561']	<class 'django.db.utils.DatabaseError'>
['19.1679', '72.84547', '19.16866', '72.84558', '562']	<class 'django.db.utils.DatabaseError'>
['19.16057', '72.83478', '', '', '969']	<class 'django.db.utils.DatabaseError'>
['19.16387', '72.83472', '', '', '970']	<class 'django.db.utils.DatabaseError'>
['19.16876', '72.83533', '', '', '971']	<class 'django.db.utils.DatabaseError'>
['19.15442', '72.83958', '19.15441', '72.83957', '1219']	<class 'django.db.utils.DatabaseError'>
['19.15912', '72.84051', '19.15915', '72.84075', '1221']	<class 'django.db.utils.DatabaseError'>
['19.15897', '72.84306', '', '', '1222']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1308']	<class 'django.db.utils.DatabaseError'>
['19.15897', '72.84306', '19.15907', '72.84331', '1419']	<class 'django.db.utils.DatabaseError'>
['19.16411', '72.83495', '19.16411', '72.83495', '1420']	<class 'django.db.utils.DatabaseError'>
['19.16345', '72.84395', '19.16345', '72.84395', '1455']	<class 'django.db.utils.DatabaseError'>
['19.1633', '72.84179', '19.1633', '72.84179', '1456']	<class 'django.db.utils.DatabaseError'>
['19.16324', '72.83948', '19.16324', '72.83948', '1457']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1459']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1460']	<class 'django.db.utils.DatabaseError'>
['19.16876', '72.83533', '19.16876', '72.83533', '1461']	<class 'django.db.utils.DatabaseError'>
['19.16602', '72.83485', '19.16632', '72.83512', '1462']	<class 'django.db.utils.DatabaseError'>
['19.15715', '72.83504', '', '', '2173']	<class 'django.db.utils.DatabaseError'>
['19.16349', '72.83578', '19.16349', '72.83578', '2949']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2950']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2951']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2952']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2953']	<class 'django.db.utils.DatabaseError'>
['19.17747', '72.83707', '', '', '2987']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3386']	<class 'django.db.utils.DatabaseError'>
['19.15173', '72.84687', '', '', '3387']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3388']	<class 'django.db.utils.DatabaseError'>
['19.17022', '72.83524', '', '', '3646']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3677']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3680']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3683']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4104']	<class 'django.db.utils.DatabaseError'>
['19.05542', '72.91054', '', '', '341']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1715']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3983']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3987']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '342']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1086']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2627']	<class 'django.db.utils.DatabaseError'>
['18.95458', '72.81702', '18.95458', '72.81702', '514']	<class 'django.db.utils.DatabaseError'>
['18.96024', '72.81732', '18.9602', '72.8171', '696']	<class 'django.db.utils.DatabaseError'>
['18.96024', '72.81732', '18.96024', '72.81732', '807']	<class 'django.db.utils.DatabaseError'>
['18.96472', '72.81837', '18.96472', '72.81837', '822']	<class 'django.db.utils.DatabaseError'>
['18.96183', '72.81438', '18.96196', '72.81444', '1049']	<class 'django.db.utils.DatabaseError'>
['18.96531', '72.81961', '18.96498', '72.81978', '1133']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1141']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1142']	<class 'django.db.utils.DatabaseError'>
['18.96149', '72.81987', '18.96149', '72.81987', '']	<type 'exceptions.ValueError'>
['', '', '', '', '1148']	<class 'django.db.utils.DatabaseError'>
['18.98222', '72.81461', '18.98194', '72.81444', '448']	<class 'django.db.utils.DatabaseError'>
['18.98', '72.81278', '18.98', '72.81278', '993']	<class 'django.db.utils.DatabaseError'>
['18.97855', '72.81389', '18.97835', '72.81354', '993']	<class 'django.db.utils.DatabaseError'>
['19.11633', '72.83124', '', '', '438']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '612']	<class 'django.db.utils.DatabaseError'>
['19.12077', '72.82644', '19.12124', '72.82623', '613']	<class 'django.db.utils.DatabaseError'>
['19.12325', '72.82432', '', '', '614']	<class 'django.db.utils.DatabaseError'>
['19.10787', '72.82989', '', '', '1277']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1399']	<class 'django.db.utils.DatabaseError'>
['19.11473', '72.82794', '19.11464', '72.82827', '2540']	<class 'django.db.utils.DatabaseError'>
['19.12847', '72.85511', '19.12948', '72.85541', '638']	<class 'django.db.utils.DatabaseError'>
['19.13538', '72.8549', '19.1345', '72.85538', '639']	<class 'django.db.utils.DatabaseError'>
['19.13936', '72.85465', '19.13961', '72.85509', '640']	<class 'django.db.utils.DatabaseError'>
['19.14146', '72.85458', '19.14635', '72.85634', '641']	<class 'django.db.utils.DatabaseError'>
['19.13928', '72.86524', '19.13923', '72.86582', '1172']	<class 'django.db.utils.DatabaseError'>
['19.12998', '72.86839', '19.12953', '72.86826', '1195']	<class 'django.db.utils.DatabaseError'>
['19.13226', '72.86962', '19.13278', '72.86976', '1196']	<class 'django.db.utils.DatabaseError'>
['19.13607', '72.87016', '19.13591', '72.87008', '1197']	<class 'django.db.utils.DatabaseError'>
['19.13807', '72.87209', '19.13823', '72.87226', '1198']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1199']	<class 'django.db.utils.DatabaseError'>
['19.13973', '72.86915', '19.13964', '72.8691', '1200']	<class 'django.db.utils.DatabaseError'>
['19.13948', '72.86077', '19.13918', '72.86149', '1385']	<class 'django.db.utils.DatabaseError'>
['19.14094', '72.85623', '19.14092', '72.85653', '1386']	<class 'django.db.utils.DatabaseError'>
['19.1298', '72.84989', '', '', '1463']	<class 'django.db.utils.DatabaseError'>
['19.1268', '72.85518', '', '', '1464']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1465']	<class 'django.db.utils.DatabaseError'>
['19.13561', '72.85052', '', '', '1466']	<class 'django.db.utils.DatabaseError'>
['19.1368', '72.85335', '', '', '1468']	<class 'django.db.utils.DatabaseError'>
['19.13783', '72.85638', '', '', '1469']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1470']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1482']	<class 'django.db.utils.DatabaseError'>
['', '', '19.13875', '72.86353', '1483']	<class 'django.db.utils.DatabaseError'>
['19.12643', '72.86373', '', '', '1632']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1633']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1930']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1932']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1933']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1934']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2006']	<class 'django.db.utils.DatabaseError'>
['19.13387', '72.87009', '19.13402', '72.86925', '2287']	<class 'django.db.utils.DatabaseError'>
['19.14049', '72.85857', '19.14029', '72.85872', '2399']	<class 'django.db.utils.DatabaseError'>
['19.13907', '72.86364', '', '', '2450']	<class 'django.db.utils.DatabaseError'>
['19.14623', '72.85575', '19.14635', '72.85635', '3002']	<class 'django.db.utils.DatabaseError'>
['19.13681', '72.85335', '', '', '3429']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3441']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3591']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3593']	<class 'django.db.utils.DatabaseError'>
['19.13832', '72.8707', '19.13845', '72.87074', '3596']	<class 'django.db.utils.DatabaseError'>
['19.13412', '72.86651', '', '', '3614']	<class 'django.db.utils.DatabaseError'>
['19.12959', '72.86501', '', '', '3628']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3994']	<class 'django.db.utils.DatabaseError'>
['19.13992', '72.86774', '19.13969', '72.86732', '']	<type 'exceptions.ValueError'>
['19.13388', '72.84782', '', '', '122']	<class 'django.db.utils.DatabaseError'>
['19.14091', '72.84319', '19.14157', '72.84337', '123']	<class 'django.db.utils.DatabaseError'>
['19.14416', '72.84207', '', '', '291']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '292']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '293']	<class 'django.db.utils.DatabaseError'>
['19.14685', '72.83669', '19.14685', '72.83669', '295']	<class 'django.db.utils.DatabaseError'>
['19.13146', '72.84769', '19.13382', '72.84115', '555']	<class 'django.db.utils.DatabaseError'>
['19.13836', '72.84524', '19.13794', '72.84594', '556']	<class 'django.db.utils.DatabaseError'>
['19.15085', '72.83152', '', '', '1407']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1427']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3705']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3706']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3707']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3708']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3709']	<class 'django.db.utils.DatabaseError'>
['19.0859', '72.82796', '19.08599', '72.82813', '134']	<class 'django.db.utils.DatabaseError'>
['19.09008', '72.82746', '19.09008', '72.82746', '135']	<class 'django.db.utils.DatabaseError'>
['19.09221', '72.82793', '19.09221', '72.82793', '136']	<class 'django.db.utils.DatabaseError'>
['19.09542', '72.82822', '19.09542', '72.82822', '137']	<class 'django.db.utils.DatabaseError'>
['19.10346', '72.82692', '19.10346', '72.82692', '468']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '469']	<class 'django.db.utils.DatabaseError'>
['19.0997', '72.82757', '19.0997', '72.82757', '470']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '471']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '507']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '518']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '520']	<class 'django.db.utils.DatabaseError'>
['19.11567', '72.83494', '', '', '527']	<class 'django.db.utils.DatabaseError'>
['19.11805', '72.82877', '19.11805', '72.82877', '528']	<class 'django.db.utils.DatabaseError'>
['19.10368', '72.83201', '', '', '1276']	<class 'django.db.utils.DatabaseError'>
['19.11363', '72.83687', '', '', '1280']	<class 'django.db.utils.DatabaseError'>
['19.08539', '72.83042', '19.08539', '72.83042', '1398']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1717']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2099']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2100']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2101']	<class 'django.db.utils.DatabaseError'>
['19.12539', '72.82242', '19.12539', '72.82242', '2862']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3717']	<class 'django.db.utils.DatabaseError'>
['19.08832', '72.82721', '', '', '3975']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3910']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3911']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3912']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3913']	<class 'django.db.utils.DatabaseError'>
['18.98466', '72.8422', '18.9846', '72.84231', '273']	<class 'django.db.utils.DatabaseError'>
['18.98745', '72.84152', '18.9874', '72.84144', '710']	<class 'django.db.utils.DatabaseError'>
['18.98808', '72.83926', '18.98808', '72.83926', '711']	<class 'django.db.utils.DatabaseError'>
['18.98815', '72.84112', '18.98811', '72.84202', '722']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '869']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '870']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2131']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2132']	<class 'django.db.utils.DatabaseError'>
['', '', '18.97967', '72.84347', '2744']	<class 'django.db.utils.DatabaseError'>
['18.98765', '72.83567', '18.9899', '72.83573', '3475']	<class 'django.db.utils.DatabaseError'>
['18.98763', '72.83567', '18.98951', '72.83602', '3475']	<class 'django.db.utils.DatabaseError'>
['18.98837', '72.83661', '', '', '3475']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3856']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3857']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3858']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3859']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3860']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3861']	<class 'django.db.utils.DatabaseError'>
['19.07785', '72.86193', '19.07741', '72.86207', '604']	<class 'django.db.utils.DatabaseError'>
['19.07336', '72.86092', '', '', '605']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '606']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '608']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1641']	<class 'django.db.utils.DatabaseError'>
['19.08067', '72.85898', '19.08061', '72.85894', '1643']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2129']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3627']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3629']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3630']	<class 'django.db.utils.DatabaseError'>
['', '', '19.07384', '72.86923', '3631']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3633']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3635']	<class 'django.db.utils.DatabaseError'>
['19.0740361111111', '72.8520722222222', '', '', '3812']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3817']	<class 'django.db.utils.DatabaseError'>
['', '', '19.07491', '72.86063', '3818']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3990']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4124']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4125']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4149']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4015']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4016']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4017']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4018']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4146']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4151']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4152']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3852']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3853']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3854']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3855']	<class 'django.db.utils.DatabaseError'>
['19.19876', '72.86072', '19.19929', '72.86099', '655']	<class 'django.db.utils.DatabaseError'>
['19.2113', '72.86827', '19.21174', '72.86876', '657']	<class 'django.db.utils.DatabaseError'>
['19.20205', '72.85407', '', '', '1539']	<class 'django.db.utils.DatabaseError'>
['19.20185', '72.85639', '', '', '1540']	<class 'django.db.utils.DatabaseError'>
['19.20158', '72.8605', '19.20158', '72.8605', '1541']	<class 'django.db.utils.DatabaseError'>
['19.20062', '72.8633', '19.20096', '72.86366', '1543']	<class 'django.db.utils.DatabaseError'>
['19.20128', '72.86634', '19.2013', '72.86613', '1544']	<class 'django.db.utils.DatabaseError'>
['19.20073', '72.8709', '19.20067', '72.8716', '1545']	<class 'django.db.utils.DatabaseError'>
['19.1992', '72.87307', '', '', '1546']	<class 'django.db.utils.DatabaseError'>
['', '', '19.19757', '72.87214', '1547']	<class 'django.db.utils.DatabaseError'>
['19.19711', '72.87505', '', '', '1548']	<class 'django.db.utils.DatabaseError'>
['19.20008', '72.87833', '19.20013', '72.87823', '1550']	<class 'django.db.utils.DatabaseError'>
['19.20109', '72.87891', '19.20453', '72.87845', '1551']	<class 'django.db.utils.DatabaseError'>
['19.20106', '72.87881', '', '', '1552']	<class 'django.db.utils.DatabaseError'>
['19.20851', '72.86416', '', '', '1602']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1603']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1734']	<class 'django.db.utils.DatabaseError'>
['19.2131', '72.86327', '', '', '2390']	<class 'django.db.utils.DatabaseError'>
['19.20893', '72.86316', '19.20893', '72.86316', '2391']	<class 'django.db.utils.DatabaseError'>
['19.21012', '72.86319', '', '', '2392']	<class 'django.db.utils.DatabaseError'>
['19.19711', '72.87505', '', '', '2835']	<class 'django.db.utils.DatabaseError'>
['19.20124', '72.86491', '19.20114', '72.86483', '2850']	<class 'django.db.utils.DatabaseError'>
['19.2099', '72.87298', '19.21003', '72.87305', '2873']	<class 'django.db.utils.DatabaseError'>
['19.2062', '72.87256', '19.2062', '72.87273', '2874']	<class 'django.db.utils.DatabaseError'>
['19.20567', '72.86911', '19.20587', '72.86913', '2875']	<class 'django.db.utils.DatabaseError'>
['19.20828', '72.87302', '19.20828', '72.87302', '2963']	<class 'django.db.utils.DatabaseError'>
['19.2011', '72.87891', '', '', '3003']	<class 'django.db.utils.DatabaseError'>
['19.19506', '72.87563', '', '', '3004']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3178']	<class 'django.db.utils.DatabaseError'>
['19.21191', '72.87323', '', '', '3180']	<class 'django.db.utils.DatabaseError'>
['19.214412', '72.86697', '', '', '3419']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3705']	<class 'django.db.utils.DatabaseError'>
['19.1944', '72.86828', '', '', '3809']	<class 'django.db.utils.DatabaseError'>
['19.20128', '72.86634', '', '', '3810']	<class 'django.db.utils.DatabaseError'>
['19.19717', '72.86999', '', '', '3811']	<class 'django.db.utils.DatabaseError'>
['19.21227', '72.86757', '', '', '3863']	<class 'django.db.utils.DatabaseError'>
['19.20964', '72.86754', '', '', '3864']	<class 'django.db.utils.DatabaseError'>
['19.20775', '72.86632', '', '', '3865']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3957']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4027']	<class 'django.db.utils.DatabaseError'>
['19.2045', '72.82839', '', '', '974']	<class 'django.db.utils.DatabaseError'>
['19.19724', '72.84763', '19.19615', '72.84753', '1227']	<class 'django.db.utils.DatabaseError'>
['', '', '19.2', '72.84876', '1228']	<class 'django.db.utils.DatabaseError'>
['19.20573', '72.85007', '', '', '1229']	<class 'django.db.utils.DatabaseError'>
['19.21375', '72.85069', '19.2137', '72.85099', '1230']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1231']	<class 'django.db.utils.DatabaseError'>
['19.22167', '72.85307', '19.21999', '72.85251', '1232']	<class 'django.db.utils.DatabaseError'>
['19.21122', '72.85153', '19.21123', '72.85139', '1290']	<class 'django.db.utils.DatabaseError'>
['19.20842', '72.85048', '19.20844', '72.85071', '1297']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1322']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1323']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1324']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1325']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1326']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1327']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1328']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1562']	<class 'django.db.utils.DatabaseError'>
['', '', '19.21526', '72.83703', '1568']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1569']	<class 'django.db.utils.DatabaseError'>
['19.20031', '72.87692', '', '', '1583']	<class 'django.db.utils.DatabaseError'>
['19.21288', '72.84336', '19.21275', '72.84316', '2158']	<class 'django.db.utils.DatabaseError'>
['19.21442', '72.83952', '19.21441', '72.83938', '2281']	<class 'django.db.utils.DatabaseError'>
['19.21231', '72.84685', '19.21219', '72.84686', '2719']	<class 'django.db.utils.DatabaseError'>
['19.21319', '72.84122', '19.21354', '72.84138', '2720']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2723']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2724']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2772']	<class 'django.db.utils.DatabaseError'>
['19.20203', '72.84893', '19.20209', '72.84914', '2849']	<class 'django.db.utils.DatabaseError'>
['19.21533', '72.83327', '19.21534', '72.82427', '2986']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3297']	<class 'django.db.utils.DatabaseError'>
['19.21379', '72.83692', '19.21345', '72.83693', '3298']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3584']	<class 'django.db.utils.DatabaseError'>
['19.20863', '72.83496', '19.2087', '72.83531', '3611']	<class 'django.db.utils.DatabaseError'>
['19.20374', '72.83411', '19.20387', '72.83438', '3613']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3617']	<class 'django.db.utils.DatabaseError'>
['19.20629', '72.83465', '', '', '3619']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3686']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3687']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3698']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3780']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3781']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3989']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4028']	<class 'django.db.utils.DatabaseError'>
['19.23447', '72.86616', '', '', '4029']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4030']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4031']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4032']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4066']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4067']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4068']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4069']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4070']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4081']	<class 'django.db.utils.DatabaseError'>
['19.21234', '72.83242', '19.21234', '72.83242', '4082']	<class 'django.db.utils.DatabaseError'>
['19.21061', '72.83041', '19.21065', '72.8305', '4083']	<class 'django.db.utils.DatabaseError'>
['19.21507', '72.81613', '19.21523', '72.81611', '4119']	<class 'django.db.utils.DatabaseError'>
['19.21351', '72.8332', '19.21341', '72.83331', '4120']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '']	<type 'exceptions.ValueError'>
['19.21701', '72.86674', '', '', '658']	<class 'django.db.utils.DatabaseError'>
['19.20546', '72.86559', '', '', '1572']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2964']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '389']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1261']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3950']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '406']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '407']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '408']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2027']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1008']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1009']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1010']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2057']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2750']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3571']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4092']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4093']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4094']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4095']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4096']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4097']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '109']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '517']	<class 'django.db.utils.DatabaseError'>
['19.06784', '72.84641', '19.06831', '72.84679', '64']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '110']	<class 'django.db.utils.DatabaseError'>
['19.06871', '72.8336', '19.06941', '72.83349', '459']	<class 'django.db.utils.DatabaseError'>
['', '', '19.07071', '72.83716', '460']	<class 'django.db.utils.DatabaseError'>
['19.07613', '72.83423', '19.07499', '72.83367', '461']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '525']	<class 'django.db.utils.DatabaseError'>
['19.07127', '72.82207', '', '', '1095']	<class 'django.db.utils.DatabaseError'>
['19.07487', '72.823', '19.07441', '72.82266', '1097']	<class 'django.db.utils.DatabaseError'>
['19.07502', '72.82616', '19.07482', '72.82662', '1098']	<class 'django.db.utils.DatabaseError'>
['19.07327', '72.83073', '19.07312', '72.83084', '1100']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1102']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1348']	<class 'django.db.utils.DatabaseError'>
['19.07388', '72.82911', '19.0739', '72.82909', '2937']	<class 'django.db.utils.DatabaseError'>
['19.07244', '72.83282', '19.07235', '72.833', '2938']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3848']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3849']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3850']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3851']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3934']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3935']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3936']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3937']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3938']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3939']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3986']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4019']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4020']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4021']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4022']	<class 'django.db.utils.DatabaseError'>
['19.02365', '72.85202', '', '', '148']	<class 'django.db.utils.DatabaseError'>
['19.02678', '72.85468', '', '', '149']	<class 'django.db.utils.DatabaseError'>
['19.03259', '72.85856', '', '', '150']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '215']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '216']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '852']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3035']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1886']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1887']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1888']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1889']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2645']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2991']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3001']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3959']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4127']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4128']	<class 'django.db.utils.DatabaseError'>
['19.05504', '72.88244', '19.05527', '72.88271', '161']	<class 'django.db.utils.DatabaseError'>
['19.05837', '72.88274', '', '', '162']	<class 'django.db.utils.DatabaseError'>
['19.06005', '72.88339', '', '', '163']	<class 'django.db.utils.DatabaseError'>
['19.05966', '72.88541', '19.05954', '72.88547', '164']	<class 'django.db.utils.DatabaseError'>
['19.06169', '72.88364', '19.06194', '72.88368', '165']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '169']	<class 'django.db.utils.DatabaseError'>
['19.05764', '72.88813', '', '', '319']	<class 'django.db.utils.DatabaseError'>
['19.05778', '72.88672', '', '', '1761']	<class 'django.db.utils.DatabaseError'>
['19.05437', '72.88155', '', '', '2286']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '238']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '239']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '240']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '241']	<class 'django.db.utils.DatabaseError'>
['19.06783', '72.87884', '', '', '242']	<class 'django.db.utils.DatabaseError'>
['19.08183', '72.88171', '19.08222', '72.88204', '243']	<class 'django.db.utils.DatabaseError'>
['19.08555', '72.88138', '19.08615', '72.88103', '360']	<class 'django.db.utils.DatabaseError'>
['', '72.8831', '', '', '361']	<class 'django.db.utils.DatabaseError'>
['19.07533', '72.87869', '', '', '590']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '607']	<class 'django.db.utils.DatabaseError'>
['19.07338', '72.88031', '19.07338', '72.88031', '610']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '611']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '842']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1247']	<class 'django.db.utils.DatabaseError'>
['19.0773', '72.87856', '19.0773', '72.87856', '1248']	<class 'django.db.utils.DatabaseError'>
['19.079', '72.87966', '19.07904', '72.87987', '1249']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1639']	<class 'django.db.utils.DatabaseError'>
['19.08065', '72.87259', '19.08065', '72.87259', '1640']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1678']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1790']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2128']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4049']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4050']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4051']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4052']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4053']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4054']	<class 'django.db.utils.DatabaseError'>
['18.99274', '72.83642', '18.99314', '72.83654', '22']	<class 'django.db.utils.DatabaseError'>
['18.99274', '72.83642', '18.99212', '72.8364', '22']	<class 'django.db.utils.DatabaseError'>
['18.99196', '72.83636', '18.99196', '72.83636', '22']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '808']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3570']	<class 'django.db.utils.DatabaseError'>
['18.99695', '72.83121', '18.99752', '72.83131', '56']	<class 'django.db.utils.DatabaseError'>
['18.99714', '72.83118', '18.99684', '72.8314', '56']	<class 'django.db.utils.DatabaseError'>
['18.9897', '72.82841', '18.99825', '72.82818', '713']	<class 'django.db.utils.DatabaseError'>
['18.99848', '72.82511', '18.99852', '72.82522', '714']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '775']	<class 'django.db.utils.DatabaseError'>
['18.99774', '72.82709', '', '', '781']	<class 'django.db.utils.DatabaseError'>
['18.99537', '72.82576', '18.99773', '72.82709', '']	<type 'exceptions.ValueError'>
['19.14457', '72.80187', '', '', '1494']	<class 'django.db.utils.DatabaseError'>
['19.19659', '72.81611', '19.19659', '72.81654', '1496']	<class 'django.db.utils.DatabaseError'>
['19.16151', '72.78637', '', '', '1508']	<class 'django.db.utils.DatabaseError'>
['19.15189', '72.79065', '19.15227', '72.79027', '1511']	<class 'django.db.utils.DatabaseError'>
['19.14424', '72.7903', '19.14473', '72.79026', '1512']	<class 'django.db.utils.DatabaseError'>
['19.14123', '72.78998', '19.14159', '72.78984', '1513']	<class 'django.db.utils.DatabaseError'>
['19.14063', '72.79121', '19.14068', '72.79102', '1514']	<class 'django.db.utils.DatabaseError'>
['19.14264', '72.7978', '19.14285', '72.79848', '1515']	<class 'django.db.utils.DatabaseError'>
['19.14819', '72.79077', '19.14773', '72.79071', '2615']	<class 'django.db.utils.DatabaseError'>
['19.1415', '72.79407', '19.14159', '72.79456', '2616']	<class 'django.db.utils.DatabaseError'>
['19.14202', '72.79568', '19.1419', '72.79639', '3772']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1598']	<class 'django.db.utils.DatabaseError'>
['18.98328', '72.8242', '18.98337', '72.82442', '697']	<class 'django.db.utils.DatabaseError'>
['', '', '18.97648', '72.80831', '']	<type 'exceptions.ValueError'>
['18.97975', '72.82031', '18.97975', '72.82031', '']	<type 'exceptions.ValueError'>
['18.98324', '72.82414', '18.98301', '72.82423', '697']	<class 'django.db.utils.DatabaseError'>
['18.98339', '72.82452', '18.98339', '72.82452', '697']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1128']	<class 'django.db.utils.DatabaseError'>
['18.97894', '72.81559', '18.97861', '72.81501', '1149']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1892']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1893']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1895']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2702']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2767']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2768']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3143']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4085']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4086']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4087']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4088']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4089']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4090']	<class 'django.db.utils.DatabaseError'>
['18.94571', '72.83404', '', '', '11']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '678']	<class 'django.db.utils.DatabaseError'>
['19.03371', '72.84235', '19.03445', '72.84231', '33']	<class 'django.db.utils.DatabaseError'>
['19.03402', '72.84225', '19.03349', '72.84221', '33']	<class 'django.db.utils.DatabaseError'>
['19.03414', '72.8422', '', '', '33']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '36']	<class 'django.db.utils.DatabaseError'>
['19.04112', '72.84086', '19.04143', '72.84086', '44']	<class 'django.db.utils.DatabaseError'>
['19.03969', '72.84154', '19.04104', '72.84098', '44']	<class 'django.db.utils.DatabaseError'>
['', '', '19.04037', '72.84102', '44']	<class 'django.db.utils.DatabaseError'>
['19.04012', '72.84161', '19.04012', '72.84161', '44']	<class 'django.db.utils.DatabaseError'>
['19.04618', '72.84346', '19.04616', '72.84353', '728']	<class 'django.db.utils.DatabaseError'>
['19.03968', '72.84169', '19.04143', '72.84086', '777']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '835']	<class 'django.db.utils.DatabaseError'>
['19.04455', '72.84455', '', '', '931']	<class 'django.db.utils.DatabaseError'>
['19.03408', '72.83861', '19.05644', '72.84679', '1063']	<class 'django.db.utils.DatabaseError'>
['19.04598', '72.83838', '19.04658', '72.83851', '2449']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2794']	<class 'django.db.utils.DatabaseError'>
['19.03797', '72.842', '19.03953', '72.84163', '3231']	<class 'django.db.utils.DatabaseError'>
['19.03723', '72.84235', '19.03723', '72.84235', '3231']	<class 'django.db.utils.DatabaseError'>
['19.04564', '72.84218', '', '', '3340']	<class 'django.db.utils.DatabaseError'>
['', '', '19.04292', '72.84557', '']	<type 'exceptions.ValueError'>
['19.03457', '72.84781', '19.03422', '72.8477', '913']	<class 'django.db.utils.DatabaseError'>
['', '', '19.04296', '72.84169', '']	<type 'exceptions.ValueError'>
['', '', '', '', '192']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '193']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '194']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '195']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '196']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1805']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2568']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3335']	<class 'django.db.utils.DatabaseError'>
['19.17951', '72.85815', '', '', '652']	<class 'django.db.utils.DatabaseError'>
['19.18002', '72.85952', '', '', '652']	<class 'django.db.utils.DatabaseError'>
['19.18974', '72.86662', '19.19159', '72.86636', '653']	<class 'django.db.utils.DatabaseError'>
['19.18776', '72.8583', '', '', '654']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1535']	<class 'django.db.utils.DatabaseError'>
['19.18847', '72.85398', '', '', '1536']	<class 'django.db.utils.DatabaseError'>
['19.18362', '72.85039', '', '', '1537']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1737']	<class 'django.db.utils.DatabaseError'>
['19.18077', '72.8532', '19.18077', '72.8532', '1739']	<class 'django.db.utils.DatabaseError'>
['19.17866', '72.85126', '19.17866', '72.85126', '1740']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1741']	<class 'django.db.utils.DatabaseError'>
['19.1874', '72.85029', '', '', '1743']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1949']	<class 'django.db.utils.DatabaseError'>
['19.18346', '72.85244', '', '', '1997']	<class 'django.db.utils.DatabaseError'>
['19.18569', '72.85914', '19.18577', '72.85962', '1998']	<class 'django.db.utils.DatabaseError'>
['', '', '19.18633', '72.86042', '1999']	<class 'django.db.utils.DatabaseError'>
['19.18048', '72.85438', '19.1803', '72.85437', '2000']	<class 'django.db.utils.DatabaseError'>
['19.18556', '72.86381', '19.18633', '72.86043', '2001']	<class 'django.db.utils.DatabaseError'>
['19.1856', '72.86382', '19.18567', '72.86371', '2002']	<class 'django.db.utils.DatabaseError'>
['19.18416', '72.86365', '', '', '2003']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2004']	<class 'django.db.utils.DatabaseError'>
['19.1876', '72.86333', '', '', '2089']	<class 'django.db.utils.DatabaseError'>
['19.18701', '72.86258', '19.188', '72.86562', '2090']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2091']	<class 'django.db.utils.DatabaseError'>
['19.18971', '72.86659', '19.1897', '72.86665', '2092']	<class 'django.db.utils.DatabaseError'>
['19.17698', '72.86971', '', '', '2160']	<class 'django.db.utils.DatabaseError'>
['', '', '19.18462', '72.85014', '2161']	<class 'django.db.utils.DatabaseError'>
['19.18146', '72.85739', '', '', '2163']	<class 'django.db.utils.DatabaseError'>
['19.17822', '72.86852', '', '', '2165']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2784']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2785']	<class 'django.db.utils.DatabaseError'>
['19.19353', '72.86587', '', '', '2786']	<class 'django.db.utils.DatabaseError'>
['19.18004', '72.86243', '19.1792', '72.86252', '2799']	<class 'django.db.utils.DatabaseError'>
['19.18409', '72.86394', '19.18416', '72.86391', '2809']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2896']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3640']	<class 'django.db.utils.DatabaseError'>
['19.19495', '72.86565', '', '', '3965']	<class 'django.db.utils.DatabaseError'>
['19.17735', '72.8459', '', '', '564']	<class 'django.db.utils.DatabaseError'>
['19.18031', '72.83655', '', '', '565']	<class 'django.db.utils.DatabaseError'>
['19.18943', '72.83556', '19.18943', '72.83556', '566']	<class 'django.db.utils.DatabaseError'>
['19.19362', '72.83436', '19.19362', '72.83436', '567']	<class 'django.db.utils.DatabaseError'>
['19.19565', '72.83508', '19.1945', '72.834', '568']	<class 'django.db.utils.DatabaseError'>
['19.19627', '72.82687', '19.1964', '72.82696', '569']	<class 'django.db.utils.DatabaseError'>
['19.17509', '72.83728', '19.17509', '72.83728', '972']	<class 'django.db.utils.DatabaseError'>
['19.18274', '72.84714', '', '', '1224']	<class 'django.db.utils.DatabaseError'>
['19.19287', '72.84714', '', '', '1226']	<class 'django.db.utils.DatabaseError'>
['19.17538', '72.84529', '19.17538', '72.84529', '1285']	<class 'django.db.utils.DatabaseError'>
['19.18082', '72.84691', '', '', '1286']	<class 'django.db.utils.DatabaseError'>
['19.19611', '72.82465', '', '', '1309']	<class 'django.db.utils.DatabaseError'>
['19.20099', '72.82804', '', '', '1316']	<class 'django.db.utils.DatabaseError'>
['19.19633', '72.80835', '19.19643', '72.80871', '1335']	<class 'django.db.utils.DatabaseError'>
['19.19213', '72.84421', '19.19213', '72.84421', '1336']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1428']	<class 'django.db.utils.DatabaseError'>
['19.19262', '72.84167', '', '', '1429']	<class 'django.db.utils.DatabaseError'>
['19.18981', '72.84878', '', '', '1493']	<class 'django.db.utils.DatabaseError'>
['19.19637', '72.81847', '19.19628', '72.81878', '1495']	<class 'django.db.utils.DatabaseError'>
['19.17649', '72.79755', '19.17654', '72.79744', '1504']	<class 'django.db.utils.DatabaseError'>
['19.17304', '72.79721', '19.17369', '72.79731', '1505']	<class 'django.db.utils.DatabaseError'>
['19.19893', '72.79885', '19.19901', '72.79875', '1516']	<class 'django.db.utils.DatabaseError'>
['19.18786', '72.84242', '', '', '1517']	<class 'django.db.utils.DatabaseError'>
['19.19163', '72.84581', '', '', '1518']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1520']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1524']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1526']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2084']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2085']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2086']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2087']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2088']	<class 'django.db.utils.DatabaseError'>
['19.179', '72.86441', '', '', '2164']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2516']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2517']	<class 'django.db.utils.DatabaseError'>
['19.15388', '72.78917', '19.15368', '72.78929', '2614']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2665']	<class 'django.db.utils.DatabaseError'>
['19.19617', '72.82465', '19.19617', '72.82469', '2998']	<class 'django.db.utils.DatabaseError'>
['19.19582', '72.83189', '19.19582', '72.83189', '3221']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3222']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3272']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3273']	<class 'django.db.utils.DatabaseError'>
['19.18421', '72.83604', '19.18421', '72.83604', '3318']	<class 'django.db.utils.DatabaseError'>
['19.19562', '72.83273', '19.19451', '72.83398', '3327']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3575']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3576']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3577']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3597']	<class 'django.db.utils.DatabaseError'>
['19.20015', '72.83399', '', '', '3610']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3673']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3678']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3691']	<class 'django.db.utils.DatabaseError'>
['19.18613', '72.83575', '19.18613', '72.83575', '3716']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4033']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4034']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4035']	<class 'django.db.utils.DatabaseError'>
['19.17509', '72.83728', '', '', '4130']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4131']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4160']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '542']	<class 'django.db.utils.DatabaseError'>
['19.1958', '72.82154', '19.19589', '72.82158', '570']	<class 'django.db.utils.DatabaseError'>
['', '', '19.1131', '72.4918', '571']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '572']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '573']	<class 'django.db.utils.DatabaseError'>
['19.19631', '72.81061', '19.19637', '72.81101', '1497']	<class 'django.db.utils.DatabaseError'>
['19.19633', '72.80835', '', '', '1498']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3292']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3293']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3294']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3295']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3980']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3981']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3982']	<class 'django.db.utils.DatabaseError'>
['19.05188', '72.93624', '', '', '2640']	<class 'django.db.utils.DatabaseError'>
['19.04198', '72.93482', '19.04173', '72.93538', '356']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1028']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1837']	<class 'django.db.utils.DatabaseError'>
['19.04523', '72.92514', '', '', '2714']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3462']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3463']	<class 'django.db.utils.DatabaseError'>
['19.04264', '72.93041', '', '', '3948']	<class 'django.db.utils.DatabaseError'>
['18.92839', '72.82369', '18.9286', '72.82364', '50']	<class 'django.db.utils.DatabaseError'>
['18.92774', '72.82811', '18.92792', '72.82813', '130']	<class 'django.db.utils.DatabaseError'>
['18.92723', '72.82988', '', '', '139']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '311']	<class 'django.db.utils.DatabaseError'>
['18.92667', '72.82085', '', '', '804']	<class 'django.db.utils.DatabaseError'>
['18.93117', '72.82638', '', '', '988']	<class 'django.db.utils.DatabaseError'>
['18.92161', '72.82649', '18.92178', '72.82645', '1048']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2106']	<class 'django.db.utils.DatabaseError'>
['18.92571', '72.83035', '18.92567', '72.8306', '2672']	<class 'django.db.utils.DatabaseError'>
['18.92568', '72.82688', '18.92449', '72.825', '3368']	<class 'django.db.utils.DatabaseError'>
['18.92542', '72.82473', '', '', '3369']	<class 'django.db.utils.DatabaseError'>
['18.92912', '72.82333', '', '', '3650']	<class 'django.db.utils.DatabaseError'>
['18.92315', '72.82649', '', '', '3652']	<class 'django.db.utils.DatabaseError'>
['18.92301', '72.82344', '', '', '3653']	<class 'django.db.utils.DatabaseError'>
['18.92449', '72.825', '', '', '3655']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3700']	<class 'django.db.utils.DatabaseError'>
['18.94156', '72.82367', '', '', '439']	<class 'django.db.utils.DatabaseError'>
['18.94638', '72.82146', '18.9451', '72.8225', '498']	<class 'django.db.utils.DatabaseError'>
['18.94847', '72.81994', '18.94932', '72.81945', '499']	<class 'django.db.utils.DatabaseError'>
['18.95596', '72.8111', '18.95606', '72.81149', '500']	<class 'django.db.utils.DatabaseError'>
['18.94158', '72.82374', '18.94119', '72.82394', '1113']	<class 'django.db.utils.DatabaseError'>
['18.95436', '72.81486', '18.95437', '72.81457', '1117']	<class 'django.db.utils.DatabaseError'>
['18.93066', '72.82259', '', '', '']	<type 'exceptions.ValueError'>
['18.93834', '72.82688', '18.94578', '72.82694', '902']	<class 'django.db.utils.DatabaseError'>
['18.94161', '72.82635', '18.9416', '72.82636', '']	<type 'exceptions.ValueError'>
['', '', '', '', '3808']	<class 'django.db.utils.DatabaseError'>
['19.11408', '72.88002', '19.11854', '72.88035', '359']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '363']	<class 'django.db.utils.DatabaseError'>
['19.10806', '72.87936', '19.10866', '72.87765', '365']	<class 'django.db.utils.DatabaseError'>
['19.10805', '72.87978', '', '', '365']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '365']	<class 'django.db.utils.DatabaseError'>
['19.11219', '72.87919', '19.11219', '72.87919', '366']	<class 'django.db.utils.DatabaseError'>
['19.11745', '72.86338', '19.11745', '72.88338', '367']	<class 'django.db.utils.DatabaseError'>
['19.11798', '72.86564', '19.11774', '72.86429', '629']	<class 'django.db.utils.DatabaseError'>
['19.11101', '72.87208', '19.1105', '72.87296', '1167']	<class 'django.db.utils.DatabaseError'>
['19.1095', '72.87523', '19.1095', '72.87523', '1170']	<class 'django.db.utils.DatabaseError'>
['19.12451', '72.86446', '19.12432', '72.86449', '1191']	<class 'django.db.utils.DatabaseError'>
['19.12556', '72.86277', '19.12568', '72.86282', '1192']	<class 'django.db.utils.DatabaseError'>
['19.12676', '72.86702', '', '', '1194']	<class 'django.db.utils.DatabaseError'>
['19.12388', '72.87987', '', '', '1269']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1382']	<class 'django.db.utils.DatabaseError'>
['19.11978', '72.86365', '19.12022', '72.86388', '1384']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1699']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1704']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1705']	<class 'django.db.utils.DatabaseError'>
['19.12929', '72.86965', '19.12948', '72.87007', '1707']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1754']	<class 'django.db.utils.DatabaseError'>
['', '', '19.12292', '72.88693', '1935']	<class 'django.db.utils.DatabaseError'>
['19.11795', '72.88474', '19.11882', '72.88532', '1937']	<class 'django.db.utils.DatabaseError'>
['19.12157', '72.88579', '', '', '1939']	<class 'django.db.utils.DatabaseError'>
['19.12171', '72.87041', '19.12058', '72.87058', '2218']	<class 'django.db.utils.DatabaseError'>
['19.12168', '72.88001', '19.12147', '72.88006', '2444']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2605']	<class 'django.db.utils.DatabaseError'>
['19.10657', '72.87514', '19.10683', '72.87496', '3301']	<class 'django.db.utils.DatabaseError'>
['19.1171', '72.88088', '19.1171', '72.88088', '3304']	<class 'django.db.utils.DatabaseError'>
['19.11531', '72.87979', '19.11531', '72.87979', '3397']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3477']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3478']	<class 'django.db.utils.DatabaseError'>
['19.10321', '72.8884', '', '', '3479']	<class 'django.db.utils.DatabaseError'>
['19.13264', '72.87456', '', '', '3516']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3578']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3579']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3637']	<class 'django.db.utils.DatabaseError'>
['19.1234', '72.87179', '19.1234', '72.87179', '3669']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3840']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3841']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3842']	<class 'django.db.utils.DatabaseError'>
['19.19186', '72.79841', '19.19186', '72.79841', '1500']	<class 'django.db.utils.DatabaseError'>
['19.19552', '72.80004', '19.19552', '72.80004', '1521']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '13']	<class 'django.db.utils.DatabaseError'>
['19.02157', '72.85044', '', '', '147']	<class 'django.db.utils.DatabaseError'>
['19.03061', '72.85784', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '207']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '942']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '943']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '944']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '951']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3344']	<class 'django.db.utils.DatabaseError'>
['19.02998', '72.84167', '19.0296', '72.84171', '32']	<class 'django.db.utils.DatabaseError'>
['19.02713', '72.84612', '19.02386', '72.84513', '911']	<class 'django.db.utils.DatabaseError'>
['19.02998', '72.84703', '19.03139', '72.84759', '912']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1802']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1803']	<class 'django.db.utils.DatabaseError'>
['18.96906', '72.8381', '18.96882', '72.83839', '99']	<class 'django.db.utils.DatabaseError'>
['18.96864', '72.84129', '18.96844', '72.84142', '100']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '101']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '102']	<class 'django.db.utils.DatabaseError'>
['18.95798', '72.84736', '18.95812', '72.8473', '665']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '667']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '670']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '708']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '721']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '992']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1011']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2048']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2049']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2050']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2051']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2052']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2053']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2054']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2055']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2056']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2058']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2356']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2578']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2594']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2595']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2749']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2751']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2847']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2848']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3467']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3606']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3609']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3621']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3883']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3884']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3885']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3942']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3943']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3944']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3945']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3947']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3995']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3996']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3997']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3998']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3999']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4057']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4058']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4140']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '2096']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1259']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1262']	<class 'django.db.utils.DatabaseError'>
['', '', '19.09997', '72.5782', '1263']	<class 'django.db.utils.DatabaseError'>
['', '', '19.10071', '72.577', '1264']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1265']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1267']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1268']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1811']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1812']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1813']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1817']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1818']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1819']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2150']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2219']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2220']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2490']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2703']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2836']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2837']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2838']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3692']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3725']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3729']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3730']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3732']	<class 'django.db.utils.DatabaseError'>
['', '', '19.1015', '72.5757', '4026']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4073']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4074']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4075']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4076']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '']	<type 'exceptions.ValueError'>
['19.18661', '72.94628', '', '', '397']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '420']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '423']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '424']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '426']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '428']	<class 'django.db.utils.DatabaseError'>
['19.17271', '72.95665', '', '', '430']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '434']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '435']	<class 'django.db.utils.DatabaseError'>
['19.18323', '72.94825', '', '', '436']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '437']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '476']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '477']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '478']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '479']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '480']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '481']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '491']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '492']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '494']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1020']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1244']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1252']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1253']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1255']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1256']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1610']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1612']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1613']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1614']	<class 'django.db.utils.DatabaseError'>
['19.17803', '72.94349', '19.17803', '72.94349', '1859']	<class 'django.db.utils.DatabaseError'>
['19.1769', '72.93866', '19.17715', '72.93897', '1860']	<class 'django.db.utils.DatabaseError'>
['19.1746', '72.93719', '19.17453', '72.93715', '1861']	<class 'django.db.utils.DatabaseError'>
['19.17274', '72.93587', '19.1732', '72.93466', '1862']	<class 'django.db.utils.DatabaseError'>
['19.1714', '72.93405', '19.17147', '72.93387', '1863']	<class 'django.db.utils.DatabaseError'>
['19.1677', '72.93187', '19.16829', '72.93188', '1864']	<class 'django.db.utils.DatabaseError'>
['', '', '19.18274', '72.94207', '2330']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2331']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2332']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2333']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2334']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2335']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2393']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2394']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2395']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2396']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2458']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2591']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2774']	<class 'django.db.utils.DatabaseError'>
['19.1699', '72.93239', '19.17004', '72.93226', '2801']	<class 'django.db.utils.DatabaseError'>
['19.16656', '72.92978', '19.16674', '72.92986', '2805']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2845']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2851']	<class 'django.db.utils.DatabaseError'>
['19.176', '72.93767', '19.176', '72.93767', '2852']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3426']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3487']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3582']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3624']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3625']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4164']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4165']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '483']	<class 'django.db.utils.DatabaseError'>
['18.96772', '72.82001', '18.96837', '72.82014', '485']	<class 'django.db.utils.DatabaseError'>
['18.96765', '72.8202', '18.96827', '72.81996', '485']	<class 'django.db.utils.DatabaseError'>
['18.9737', '72.82261', '18.97321', '72.82255', '486']	<class 'django.db.utils.DatabaseError'>
['18.97748', '72.82403', '18.97748', '72.82403', '487']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '828']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '881']	<class 'django.db.utils.DatabaseError'>
['18.97014', '72.82128', '18.97024', '72.82114', '2668']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1120']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4147']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4148']	<class 'django.db.utils.DatabaseError'>
['18.96658', '72.83255', '18.9671', '72.83283', '16']	<class 'django.db.utils.DatabaseError'>
['18.96646', '72.83259', '18.97059', '72.83283', '17']	<class 'django.db.utils.DatabaseError'>
['18.97051', '72.83282', '18.97051', '72.83282', '17']	<class 'django.db.utils.DatabaseError'>
['18.96985', '72.83494', '18.96985', '72.8351', '98']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '754']	<class 'django.db.utils.DatabaseError'>
['18.96708', '72.82741', '18.96708', '72.82741', '755']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '880']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '884']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '885']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1125']	<class 'django.db.utils.DatabaseError'>
['18.96708', '72.82741', '18.96708', '72.82741', '1126']	<class 'django.db.utils.DatabaseError'>
['18.96892', '72.83327', '18.96873', '72.83314', '2683']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4150']	<class 'django.db.utils.DatabaseError'>
['18.96228', '72.83201', '18.96356', '72.83227', '']	<type 'exceptions.ValueError'>
['', '', '', '', '2488']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3465']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3588']	<class 'django.db.utils.DatabaseError'>
['18.96131', '72.81311', '', '', '501']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '758']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '759']	<class 'django.db.utils.DatabaseError'>
['18.95992', '72.81228', '', '', '845']	<class 'django.db.utils.DatabaseError'>
['18.95089', '72.79607', '', '', '589']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '591']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '592']	<class 'django.db.utils.DatabaseError'>
['18.95241', '72.79713', '18.95291', '72.79751', '593']	<class 'django.db.utils.DatabaseError'>
['18.95552', '72.79846', '18.95463', '72.79837', '594']	<class 'django.db.utils.DatabaseError'>
['18.96138', '72.80141', '18.96464', '72.80327', '596']	<class 'django.db.utils.DatabaseError'>
['18.9636', '72.80302', '18.96638', '72.8035', '597']	<class 'django.db.utils.DatabaseError'>
['18.96833', '72.80343', '18.96855', '72.80373', '598']	<class 'django.db.utils.DatabaseError'>
['18.97069', '72.80473', '18.9724', '72.80502', '599']	<class 'django.db.utils.DatabaseError'>
['18.97462', '72.80625', '18.97462', '72.80625', '600']	<class 'django.db.utils.DatabaseError'>
['18.97625', '72.80844', '18.97615', '72.80738', '601']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3728']	<class 'django.db.utils.DatabaseError'>
['18.9568', '72.7997', '18.95631', '72.79943', '3738']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1018']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1042']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1043']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1044']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1955']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1960']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1982']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1985']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2646']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2992']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3915']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3916']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3917']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3919']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3920']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3921']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3922']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3923']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3926']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3927']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3929']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3930']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3931']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3941']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3958']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '442']	<class 'django.db.utils.DatabaseError'>
['18.9602', '72.81711', '18.95551', '72.81628', '666']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '695']	<class 'django.db.utils.DatabaseError'>
['18.95851', '72.81967', '18.96145', '72.81994', '1109']	<class 'django.db.utils.DatabaseError'>
['18.95917', '72.81446', '18.9593', '72.81453', '3433']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '125']	<class 'django.db.utils.DatabaseError'>
['19.13186', '72.83343', '', '', '530']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '534']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '535']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '536']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '960']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '961']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '962']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '963']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '964']	<class 'django.db.utils.DatabaseError'>
['19.14712', '72.83136', '19.14831', '72.83138', '1404']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1405']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1408']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1421']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2631']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2632']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3410']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3710']	<class 'django.db.utils.DatabaseError'>
['19.14656', '72.84645', '', '', '3711']	<class 'django.db.utils.DatabaseError'>
['19.14787', '72.84814', '', '', '3712']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3955']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3956']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '24']	<class 'django.db.utils.DatabaseError'>
['19.00433', '72.84111', '19.00455', '72.84081', '25']	<class 'django.db.utils.DatabaseError'>
['19.00493', '72.84033', '', '', '25']	<class 'django.db.utils.DatabaseError'>
['19.01066', '72.8428', '19.01066', '72.8428', '26']	<class 'django.db.utils.DatabaseError'>
['19.00977', '72.84252', '19.00977', '72.84252', '26']	<class 'django.db.utils.DatabaseError'>
['19.00974', '72.84241', '19.00974', '72.84241', '26']	<class 'django.db.utils.DatabaseError'>
['19.00867', '72.84179', '19.00867', '72.84179', '26']	<class 'django.db.utils.DatabaseError'>
['18.99743', '72.84041', '18.99775', '72.84076', '203']	<class 'django.db.utils.DatabaseError'>
['19.00152', '72.8407', '19.00157', '72.8408', '204']	<class 'django.db.utils.DatabaseError'>
['19.00446', '72.84315', '19.00444', '72.84307', '232']	<class 'django.db.utils.DatabaseError'>
['19.00149', '72.84425', '19.00153', '72.84415', '545']	<class 'django.db.utils.DatabaseError'>
['18.9994', '72.83999', '', '', '823']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '824']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '846']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '898']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2967']	<class 'django.db.utils.DatabaseError'>
['19.01226', '72.83126', '', '', '3124']	<class 'django.db.utils.DatabaseError'>
['19.00265', '72.84283', '19.00279', '72.84256', '3502']	<class 'django.db.utils.DatabaseError'>
['18.96722', '72.80778', '18.96694', '72.80778', '444']	<class 'django.db.utils.DatabaseError'>
['18.96944', '72.80889', '', '72.80917', '445']	<class 'django.db.utils.DatabaseError'>
['18.97333', '72.80889', '18.97278', '72.80917', '446']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1564']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1565']	<class 'django.db.utils.DatabaseError'>
['19.21442', '72.83952', '19.21441', '72.83938', '2282']	<class 'django.db.utils.DatabaseError'>
['19.21527', '72.83574', '19.21543', '72.83581', '2721']	<class 'django.db.utils.DatabaseError'>
['19.21234', '72.84514', '19.21216', '72.84527', '2766']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3862']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3895']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '371']	<class 'django.db.utils.DatabaseError'>
['19.1257', '72.89437', '', '', '372']	<class 'django.db.utils.DatabaseError'>
['19.11492', '72.88895', '19.11581', '72.88921', '1168']	<class 'django.db.utils.DatabaseError'>
['19.12747', '72.91849', '', '', '1179']	<class 'django.db.utils.DatabaseError'>
['19.12554', '72.91689', '19.12467', '72.91594', '1180']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1181']	<class 'django.db.utils.DatabaseError'>
['19.12314', '72.8983', '19.12311', '72.89803', '1182']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1201']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1202']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1203']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1204']	<class 'django.db.utils.DatabaseError'>
['19.11969', '72.90253', '19.12018', '72.89961', '1627']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1870']	<class 'django.db.utils.DatabaseError'>
['19.12627', '72.89055', '19.12637', '72.8901', '1940']	<class 'django.db.utils.DatabaseError'>
['19.11925', '72.90378', '19.11919', '72.90386', '2275']	<class 'django.db.utils.DatabaseError'>
['19.12296', '72.91307', '19.12327', '72.91275', '2306']	<class 'django.db.utils.DatabaseError'>
['19.12059', '72.91279', '19.11968', '72.91235', '2307']	<class 'django.db.utils.DatabaseError'>
['19.11897', '72.90823', '19.11898', '72.90833', '2308']	<class 'django.db.utils.DatabaseError'>
['19.12049', '72.90655', '19.12039', '72.90688', '2526']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2620']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2621']	<class 'django.db.utils.DatabaseError'>
['19.1177', '72.90533', '19.11758', '72.90551', '2622']	<class 'django.db.utils.DatabaseError'>
['19.12929', '72.88146', '', '', '3815']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3898']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3899']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3900']	<class 'django.db.utils.DatabaseError'>
['19.12385', '72.893', '19.12379', '72.89251', '3974']	<class 'django.db.utils.DatabaseError'>
['19.11173', '72.91075', '', '', '4071']	<class 'django.db.utils.DatabaseError'>
['19.11581', '72.9119', '19.116', '72.91171', '4129']	<class 'django.db.utils.DatabaseError'>
['19.12844', '72.88733', '19.12845', '72.88729', '']	<type 'exceptions.ValueError'>
['', '', '19.12016', '72.89075', '']	<type 'exceptions.ValueError'>
['19.01433', '72.83549', '19.01421', '72.8353', '60']	<class 'django.db.utils.DatabaseError'>
['19.01423', '72.82357', '19.01407', '72.82359', '400']	<class 'django.db.utils.DatabaseError'>
['19.01237', '72.82284', '19.01294', '72.82462', '400']	<class 'django.db.utils.DatabaseError'>
['19.01241', '72.82281', '19.01283', '72.82583', '400']	<class 'django.db.utils.DatabaseError'>
['19.01412', '72.83008', '19.01412', '72.83008', '402']	<class 'django.db.utils.DatabaseError'>
['19.01358', '72.8295', '19.01358', '72.8295', '402']	<class 'django.db.utils.DatabaseError'>
['19.01336', '72.82884', '19.01336', '72.82884', '402']	<class 'django.db.utils.DatabaseError'>
['19.01329', '72.82822', '19.01329', '72.82822', '402']	<class 'django.db.utils.DatabaseError'>
['19.01412', '72.83008', '19.01283', '72.82583', '402']	<class 'django.db.utils.DatabaseError'>
['19.01801', '72.83126', '19.01885', '72.83124', '403']	<class 'django.db.utils.DatabaseError'>
['19.01575', '72.82783', '', '', '403']	<class 'django.db.utils.DatabaseError'>
['19.01612', '72.82836', '', '', '403']	<class 'django.db.utils.DatabaseError'>
['19.01869', '72.83387', '', '', '404']	<class 'django.db.utils.DatabaseError'>
['19.01359', '72.83343', '19.00147', '72.83272', '576']	<class 'django.db.utils.DatabaseError'>
['19.02294', '72.83462', '19.024', '72.83519', '2792']	<class 'django.db.utils.DatabaseError'>
['19.01467', '72.83048', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '218']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '219']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '307']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '313']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '315']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '317']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2359']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3695']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1104']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '762']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1651']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1652']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1654']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1655']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1656']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2854']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3316']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3383']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3443']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3991']	<class 'django.db.utils.DatabaseError'>
['18.97653', '72.84418', '18.97653', '72.84418', '']	<type 'exceptions.ValueError'>
['', '', '', '', '734']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '735']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1653']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1687']	<class 'django.db.utils.DatabaseError'>
['19.10388', '72.87453', '19.10381', '72.8744', '2115']	<class 'django.db.utils.DatabaseError'>
['19.10526', '72.88592', '19.10516', '72.8856', '1697']	<class 'django.db.utils.DatabaseError'>
['19.10568', '72.88515', '19.10559', '72.88524', '1697']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '676']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '677']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '687']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '688']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3906']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3907']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3908']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3909']	<class 'django.db.utils.DatabaseError'>
['19.08006', '72.84952', '19.07906', '72.84715', '79']	<class 'django.db.utils.DatabaseError'>
['19.07878', '72.84668', '19.08043', '72.84595', '79']	<class 'django.db.utils.DatabaseError'>
['19.07523', '72.84704', '19.07523', '72.84704', '577']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '578']	<class 'django.db.utils.DatabaseError'>
['19.08067', '72.85208', '19.08067', '72.85208', '602']	<class 'django.db.utils.DatabaseError'>
['19.08061', '72.85894', '19.08198', '72.85795', '603']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '645']	<class 'django.db.utils.DatabaseError'>
['19.08103', '72.84206', '', '', '1623']	<class 'django.db.utils.DatabaseError'>
['19.08158', '72.85564', '19.08178', '72.8556', '1642']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1948']	<class 'django.db.utils.DatabaseError'>
['19.08298', '72.85329', '', '', '2077']	<class 'django.db.utils.DatabaseError'>
['19.08345', '72.85215', '', '', '2078']	<class 'django.db.utils.DatabaseError'>
['19.08045', '72.84543', '19.08045', '72.84543', '2191']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2624']	<class 'django.db.utils.DatabaseError'>
['19.08098', '72.85033', '', '', '2625']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3392']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3499']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3500']	<class 'django.db.utils.DatabaseError'>
['19.07611', '72.84907', '', '', '3592']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3595']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3599']	<class 'django.db.utils.DatabaseError'>
['19.07611', '72.849', '', '', '3600']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3603']	<class 'django.db.utils.DatabaseError'>
['19.07878', '72.85138', '', '', '3605']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3616']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3622']	<class 'django.db.utils.DatabaseError'>
['19.07866', '72.84981', '', '', '3626']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3816']	<class 'django.db.utils.DatabaseError'>
['19.08696', '72.8382', '19.08696', '72.8382', '112']	<class 'django.db.utils.DatabaseError'>
['19.08701', '72.838', '19.08696', '72.83825', '112']	<class 'django.db.utils.DatabaseError'>
['19.09122', '', '19.09122', '72.83852', '113']	<class 'django.db.utils.DatabaseError'>
['19.09129', '72.83842', '19.09115', '72.83832', '113']	<class 'django.db.utils.DatabaseError'>
['19.09183', '72.83859', '', '', '113']	<class 'django.db.utils.DatabaseError'>
['19.08433', '72.8356', '', '', '132']	<class 'django.db.utils.DatabaseError'>
['19.08464', '72.83286', '19.08469', '72.83294', '467']	<class 'django.db.utils.DatabaseError'>
['19.079', '72.83433', '', '', '551']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '985']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '986']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1391']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1713']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2710']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3391']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3918']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3940']	<class 'django.db.utils.DatabaseError'>
['18.97603', '72.83196', '18.97603', '72.83196', '488']	<class 'django.db.utils.DatabaseError'>
['18.98251', '72.82793', '', '', '488']	<class 'django.db.utils.DatabaseError'>
['18.98386', '72.82942', '18.98379', '72.82923', '489']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1144']	<class 'django.db.utils.DatabaseError'>
['18.98883', '72.84574', '', '', '275']	<class 'django.db.utils.DatabaseError'>
['18.99851', '72.85335', '', '', '276']	<class 'django.db.utils.DatabaseError'>
['18.99219', '72.85016', '', '', '277']	<class 'django.db.utils.DatabaseError'>
['18.98131', '72.85371', '', '', '278']	<class 'django.db.utils.DatabaseError'>
['18.99503', '72.85359', '19.00544', '72.85382', '279']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '303']	<class 'django.db.utils.DatabaseError'>
['18.99819', '72.8525', '', '', '541']	<class 'django.db.utils.DatabaseError'>
['18.99866', '72.84966', '18.99837', '72.84973', '543']	<class 'django.db.utils.DatabaseError'>
['19.00062', '72.84605', '', '', '544']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '724']	<class 'django.db.utils.DatabaseError'>
['18.99482', '72.84884', '', '', '725']	<class 'django.db.utils.DatabaseError'>
['18.99218', '72.8502', '', '', '738']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '785']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2966']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2968']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2969']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2970']	<class 'django.db.utils.DatabaseError'>
['18.99652', '72.85306', '', '', '3267']	<class 'django.db.utils.DatabaseError'>
['18.99979', '72.85099', '', '', '3587']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1659']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3239']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3240']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '337']	<class 'django.db.utils.DatabaseError'>
['19.06237', '72.91671', '19.06257', '72.91693', '343']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1025']	<class 'django.db.utils.DatabaseError'>
['19.05347', '72.93514', '', '', '1027']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1756']	<class 'django.db.utils.DatabaseError'>
['19.05845', '72.92487', '', '', '1826']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2250']	<class 'django.db.utils.DatabaseError'>
['19.05696', '72.92783', '', '', '2251']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2842']	<class 'django.db.utils.DatabaseError'>
['19.05917', '72.91469', '', '', '2891']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2897']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3158']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3976']	<class 'django.db.utils.DatabaseError'>
['19.05632', '72.92998', '', '', '3977']	<class 'django.db.utils.DatabaseError'>
['19.06378', '72.91218', '19.06374', '72.91298', '']	<type 'exceptions.ValueError'>
['19.02621', '72.84088', '19.02584', '72.8408', '31']	<class 'django.db.utils.DatabaseError'>
['19.02388', '72.83919', '19.02431', '72.83933', '31']	<class 'django.db.utils.DatabaseError'>
['19.02674', '72.83647', '19.02698', '72.83665', '832']	<class 'django.db.utils.DatabaseError'>
['19.02876', '72.83763', '19.02878', '72.83763', '833']	<class 'django.db.utils.DatabaseError'>
['19.02069', '72.83304', '19.02171', '72.83352', '1060']	<class 'django.db.utils.DatabaseError'>
['19.03592', '72.86017', '', '', '151']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '152']	<class 'django.db.utils.DatabaseError'>
['19.04807', '72.86812', '', '', '155']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '217']	<class 'django.db.utils.DatabaseError'>
['19.04573', '72.86513', '', '', '305']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1246']	<class 'django.db.utils.DatabaseError'>
['18.96541', '72.8129', '18.965', '72.8128', '502']	<class 'django.db.utils.DatabaseError'>
['18.97028', '72.81494', '', '', '503']	<class 'django.db.utils.DatabaseError'>
['18.97307', '72.81423', '18.97743', '72.81154', '504']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '858']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '859']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '860']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3806']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3807']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3825']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3826']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3827']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3828']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4077']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4079']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4091']	<class 'django.db.utils.DatabaseError'>
['19.18789', '72.96417', '', '', '484']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '495']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3803']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3804']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3805']	<class 'django.db.utils.DatabaseError'>
['19.19167', '72.96852', '', '', '3866']	<class 'django.db.utils.DatabaseError'>
['19.19418', '72.97051', '', '', '3867']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3868']	<class 'django.db.utils.DatabaseError'>
['19.20345', '72.97186', '', '', '3869']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3870']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3871']	<class 'django.db.utils.DatabaseError'>
['19.20613', '72.98015', '', '', '3872']	<class 'django.db.utils.DatabaseError'>
['19.20601', '72.98222', '', '', '3873']	<class 'django.db.utils.DatabaseError'>
['19.20821', '72.98267', '', '', '3874']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3875']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3876']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3877']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3878']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3879']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3880']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3881']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3882']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3901']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3949']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4008']	<class 'django.db.utils.DatabaseError'>
['', '', '19.1316', '72.5923', '4009']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4010']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4011']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4012']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4013']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4014']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4099']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4100']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4101']	<class 'django.db.utils.DatabaseError'>
['19.21471', '72.97608', '', '', '4110']	<class 'django.db.utils.DatabaseError'>
['19.21631', '72.97385', '', '', '4111']	<class 'django.db.utils.DatabaseError'>
['19.21817', '72.97014', '', '', '4112']	<class 'django.db.utils.DatabaseError'>
['19.22201', '72.96982', '19.22201', '72.96994', '4114']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4115']	<class 'django.db.utils.DatabaseError'>
['19.22787', '72.96964', '', '', '4116']	<class 'django.db.utils.DatabaseError'>
['19.2284', '72.96768', '', '', '4117']	<class 'django.db.utils.DatabaseError'>
['19.22887', '72.96614', '', '', '4118']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4139']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4188']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '344']	<class 'django.db.utils.DatabaseError'>
['19.03832', '72.94481', '19.03832', '72.94481', '357']	<class 'django.db.utils.DatabaseError'>
['19.04088', '72.93965', '19.04088', '72.93965', '1057']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1058']	<class 'django.db.utils.DatabaseError'>
['19.00618', '72.82963', '19.00622', '72.82988', '887']	<class 'django.db.utils.DatabaseError'>
['18.99165', '72.82403', '18.9921', '72.82426', '3724']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1029']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1030']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1031']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1032']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1033']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1034']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1035']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1036']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1038']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1039']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1040']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1041']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1881']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1883']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1884']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1885']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1905']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1964']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1966']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1976']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1978']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1983']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1984']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2661']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2662']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2663']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2664']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2990']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3275']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3681']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3903']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2567']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3336']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3338']	<class 'django.db.utils.DatabaseError'>
['19.13957', '72.80847', '', '', '1212']	<class 'django.db.utils.DatabaseError'>
['19.13062', '72.81409', '19.13042', '72.81424', '1215']	<class 'django.db.utils.DatabaseError'>
['19.13264', '72.81239', '19.13264', '72.81239', '1216']	<class 'django.db.utils.DatabaseError'>
['19.13536', '72.80999', '19.13536', '72.80999', '1217']	<class 'django.db.utils.DatabaseError'>
['19.13839', '72.80669', '19.13839', '72.80669', '1450']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2863']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3727']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3735']	<class 'django.db.utils.DatabaseError'>
['19.07485', '72.90314', '', '', '3739']	<class 'django.db.utils.DatabaseError'>
['19.08513', '72.88747', '19.08512', '72.88515', '244']	<class 'django.db.utils.DatabaseError'>
['19.08484', '72.89034', '19.08475', '72.88985', '245']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '245']	<class 'django.db.utils.DatabaseError'>
['19.0854', '72.89332', '', '', '1250']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1628']	<class 'django.db.utils.DatabaseError'>
['19.0824', '72.89564', '', '', '1629']	<class 'django.db.utils.DatabaseError'>
['19.08195', '72.89437', '', '', '1630']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1631']	<class 'django.db.utils.DatabaseError'>
['19.08082', '72.89087', '', '', '1634']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1688']	<class 'django.db.utils.DatabaseError'>
['19.08238', '72.88578', '', '', '1689']	<class 'django.db.utils.DatabaseError'>
['19.08238', '72.88578', '', '', '4044']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4046']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4047']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '368']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '375']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '328']	<class 'django.db.utils.DatabaseError'>
['19.118056', '72.93722', '', '', '331']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '381']	<class 'django.db.utils.DatabaseError'>
['19.12556', '72.93267', '', '', '388']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1177']	<class 'django.db.utils.DatabaseError'>
['19.11371', '72.93369', '', '', '1779']	<class 'django.db.utils.DatabaseError'>
['19.11509', '72.93264', '', '', '1780']	<class 'django.db.utils.DatabaseError'>
['19.121267', '72.930658', '', '', '1781']	<class 'django.db.utils.DatabaseError'>
['19.12152', '72.93063', '', '', '1782']	<class 'django.db.utils.DatabaseError'>
['19.12388', '72.9287', '', '', '1783']	<class 'django.db.utils.DatabaseError'>
['19.14328', '72.93981', '19.11171', '72.92584', '1872']	<class 'django.db.utils.DatabaseError'>
['19.12224', '72.94033', '', '', '1876']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2293']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2607']	<class 'django.db.utils.DatabaseError'>
['19.12008', '72.94001', '', '', '2886']	<class 'django.db.utils.DatabaseError'>
['19.1186', '72.94081', '', '', '3666']	<class 'django.db.utils.DatabaseError'>
['19.1138', '72.93582', '19.11359', '72.93556', '3688']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4036']	<class 'django.db.utils.DatabaseError'>
['19.11567', '72.94071', '', '', '']	<type 'exceptions.ValueError'>
['', '', '', '', '255']	<class 'django.db.utils.DatabaseError'>
['19.10087', '72.9189', '19.10087', '72.9189', '257']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '266']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '267']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '268']	<class 'django.db.utils.DatabaseError'>
['19.10782', '72.92391', '19.1085', '72.92416', '382']	<class 'django.db.utils.DatabaseError'>
['19.11352', '72.92618', '', '', '384']	<class 'django.db.utils.DatabaseError'>
['19.11705', '72.92611', '19.11816', '72.92594', '385']	<class 'django.db.utils.DatabaseError'>
['19.12156', '72.92563', '19.12201', '72.92534', '386']	<class 'django.db.utils.DatabaseError'>
['19.12447', '72.92534', '19.12439', '72.92521', '387']	<class 'django.db.utils.DatabaseError'>
['19.1044', '72.92147', '19.10484', '72.92162', '1023']	<class 'django.db.utils.DatabaseError'>
['19.09954', '72.91764', '19.09871', '72.91715', '1251']	<class 'django.db.utils.DatabaseError'>
['19.1015', '72.91128', '', '', '1836']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1841']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1926']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2924']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2978']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2980']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2981']	<class 'django.db.utils.DatabaseError'>
['19.106944', '72.919722', '', '', '2982']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2983']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3215']	<class 'django.db.utils.DatabaseError'>
['19.11808', '72.92408', '', '', '3740']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3741']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3897']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4048']	<class 'django.db.utils.DatabaseError'>
['19.09187', '72.84639', '19.092', '72.84578', '66']	<class 'django.db.utils.DatabaseError'>
['19.09708', '72.85107', '19.09708', '72.85107', '67']	<class 'django.db.utils.DatabaseError'>
['19.09751', '72.84936', '19.09753', '72.84911', '68']	<class 'django.db.utils.DatabaseError'>
['19.10481', '72.85206', '19.10481', '72.85206', '70']	<class 'django.db.utils.DatabaseError'>
['19.10347', '72.84968', '19.10361', '72.85001', '71']	<class 'django.db.utils.DatabaseError'>
['19.10736', '72.85197', '19.10712', '72.85193', '72']	<class 'django.db.utils.DatabaseError'>
['19.10333', '72.84848', '19.10333', '72.84848', '80']	<class 'django.db.utils.DatabaseError'>
['19.09998', '72.85283', '', '', '86']	<class 'django.db.utils.DatabaseError'>
['19.09751', '72.84936', '19.09867', '72.84483', '622']	<class 'django.db.utils.DatabaseError'>
['19.09866', '72.84466', '19.10061', '72.84466', '622']	<class 'django.db.utils.DatabaseError'>
['19.10085', '72.84464', '', '', '622']	<class 'django.db.utils.DatabaseError'>
['19.10304', '72.84536', '19.10304', '72.84584', '623']	<class 'django.db.utils.DatabaseError'>
['19.09977', '72.85399', '19.10105', '72.85478', '633']	<class 'django.db.utils.DatabaseError'>
['19.10372', '72.85152', '19.10361', '72.85131', '634']	<class 'django.db.utils.DatabaseError'>
['19.10528', '72.85368', '19.10226', '72.85469', '634']	<class 'django.db.utils.DatabaseError'>
['19.10064', '72.85207', '', '', '663']	<class 'django.db.utils.DatabaseError'>
['19.09055', '72.84416', '19.09055', '72.84416', '1103']	<class 'django.db.utils.DatabaseError'>
['19.0935', '72.85585', '19.0933', '72.85272', '1471']	<class 'django.db.utils.DatabaseError'>
['19.09623', '72.85419', '', '', '1473']	<class 'django.db.utils.DatabaseError'>
['19.10283', '72.84582', '19.10283', '72.84582', '1644']	<class 'django.db.utils.DatabaseError'>
['19.10567', '72.84669', '19.10582', '72.84682', '1645']	<class 'django.db.utils.DatabaseError'>
['19.10807', '72.84856', '19.10807', '72.85856', '1646']	<class 'django.db.utils.DatabaseError'>
['19.09613', '72.84385', '19.09603', '72.84382', '1711']	<class 'django.db.utils.DatabaseError'>
['19.09411', '72.85178', '19.09216', '72.84379', '1712']	<class 'django.db.utils.DatabaseError'>
['19.09352', '72.85585', '19.09352', '72.85585', '3665']	<class 'django.db.utils.DatabaseError'>
['19.09784', '72.85354', '19.09784', '72.85354', '']	<type 'exceptions.ValueError'>
['19.09411', '72.85178', '19.09367', '72.85167', '']	<type 'exceptions.ValueError'>
['19.09505', '72.83958', '19.09511', '72.83952', '114']	<class 'django.db.utils.DatabaseError'>
['19.09621', '72.83948', '', '', '114']	<class 'django.db.utils.DatabaseError'>
['19.10227', '72.83932', '19.10234', '72.83961', '115']	<class 'django.db.utils.DatabaseError'>
['19.09942', '72.83958', '19.09942', '72.83958', '115']	<class 'django.db.utils.DatabaseError'>
['19.10739', '72.84023', '19.10699', '72.8401', '116']	<class 'django.db.utils.DatabaseError'>
['19.10364', '72.83677', '', '', '506']	<class 'django.db.utils.DatabaseError'>
['19.10435', '72.83929', '19.1046', '72.83972', '554']	<class 'django.db.utils.DatabaseError'>
['19.10368', '72.83201', '', '', '1275']	<class 'django.db.utils.DatabaseError'>
['19.11321', '72.82993', '19.11321', '72.82993', '1278']	<class 'django.db.utils.DatabaseError'>
['19.1132', '72.8297', '19.1132', '72.8297', '1279']	<class 'django.db.utils.DatabaseError'>
['19.09931', '72.83954', '', '', '1390']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4145']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '212']	<class 'django.db.utils.DatabaseError'>
['19.018842', '72.860564', '', '', '227']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '228']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '230']	<class 'django.db.utils.DatabaseError'>
['19.046608', '72.878258', '', '', '297']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '769']	<class 'django.db.utils.DatabaseError'>
['19.017692', '72.864111', '', '', '770']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '771']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1658']	<class 'django.db.utils.DatabaseError'>
['19.030667', '72.867772', '', '', '2241']	<class 'django.db.utils.DatabaseError'>
['19.020664', '72.875', '', '', '2242']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '208']	<class 'django.db.utils.DatabaseError'>
['19.011947', '72.8543083333333', '', '', '209']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '210']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '214']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '280']	<class 'django.db.utils.DatabaseError'>
['19.016111', '72.8575', '', '', '281']	<class 'django.db.utils.DatabaseError'>
['19.019278', '72.856694', '', '', '304']	<class 'django.db.utils.DatabaseError'>
['19.0149', '72.85239', '', '', '473']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '825']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '844']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1160']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1162']	<class 'django.db.utils.DatabaseError'>
['19.016111', '72.8575', '', '', '3583']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '674']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '675']	<class 'django.db.utils.DatabaseError'>
['18.95571', '72.80926', '', '', '751']	<class 'django.db.utils.DatabaseError'>
['18.94686', '72.7958', '', '', '863']	<class 'django.db.utils.DatabaseError'>
['18.94824', '72.79861', '18.94824', '72.79861', '864']	<class 'django.db.utils.DatabaseError'>
['18.94929', '72.8006', '18.94929', '72.8006', '865']	<class 'django.db.utils.DatabaseError'>
['18.95003', '72.80167', '18.95082', '72.80199', '866']	<class 'django.db.utils.DatabaseError'>
['18.95244', '72.80492', '18.9517', '72.80449', '867']	<class 'django.db.utils.DatabaseError'>
['18.95372', '72.80687', '18.95372', '72.80687', '868']	<class 'django.db.utils.DatabaseError'>
['18.95526', '72.80456', '18.95531', '72.8045', '980']	<class 'django.db.utils.DatabaseError'>
['18.95321', '72.80318', '18.95328', '72.80292', '981']	<class 'django.db.utils.DatabaseError'>
['18.94993', '72.79978', '18.94987', '72.79947', '982']	<class 'django.db.utils.DatabaseError'>
['18.94863', '72.79832', '18.94858', '72.79784', '983']	<class 'django.db.utils.DatabaseError'>
['18.9722', '72.80495', '18.9722', '72.80495', '599']	<class 'django.db.utils.DatabaseError'>
['19.01664', '72.82098', '', '', '396']	<class 'django.db.utils.DatabaseError'>
['19.01142', '72.82093', '19.01132', '72.82112', '399']	<class 'django.db.utils.DatabaseError'>
['19.01254', '72.82281', '', '', '401']	<class 'django.db.utils.DatabaseError'>
['18.98849', '72.81399', '', '', '449']	<class 'django.db.utils.DatabaseError'>
['18.98891', '72.81383', '', '', '449']	<class 'django.db.utils.DatabaseError'>
['18.98949', '72.8138', '18.98924', '72.81399', '449']	<class 'django.db.utils.DatabaseError'>
['18.98849', '72.81399', '18.98954', '72.8187', '449']	<class 'django.db.utils.DatabaseError'>
['18.98966', '72.81391', '18.98924', '72.81399', '450']	<class 'django.db.utils.DatabaseError'>
['18.9897', '72.81385', '', '', '450']	<class 'django.db.utils.DatabaseError'>
['18.98993', '72.81394', '', '', '450']	<class 'django.db.utils.DatabaseError'>
['18.98997', '72.81385', '', '', '450']	<class 'django.db.utils.DatabaseError'>
['18.99489', '72.81545', '18.9949', '72.81568', '451']	<class 'django.db.utils.DatabaseError'>
['18.99682', '72.81824', '18.99749', '72.81805', '452']	<class 'django.db.utils.DatabaseError'>
['18.99668', '72.81821', '18.99749', '72.81806', '452']	<class 'django.db.utils.DatabaseError'>
['18.99568', '72.81911', '18.99568', '72.81911', '452']	<class 'django.db.utils.DatabaseError'>
['19.00201', '72.8191', '19.00183', '72.81917', '453']	<class 'django.db.utils.DatabaseError'>
['19.00531', '72.82098', '19.00465', '72.82077', '454']	<class 'django.db.utils.DatabaseError'>
['19.00715', '72.81883', '19.00717', '72.81865', '455']	<class 'django.db.utils.DatabaseError'>
['19.0073', '72.81847', '19.00569', '72.81812', '455']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '455']	<class 'django.db.utils.DatabaseError'>
['19.00914', '72.81966', '19.00592', '72.81806', '455']	<class 'django.db.utils.DatabaseError'>
['19.00914', '72.81966', '19.00937', '72.81978', '456']	<class 'django.db.utils.DatabaseError'>
['19.00947', '72.82009', '19.00947', '72.82009', '456']	<class 'django.db.utils.DatabaseError'>
['19.0012', '72.81541', '', '', '523']	<class 'django.db.utils.DatabaseError'>
['19.00317', '72.81583', '19.00085', '72.81608', '523']	<class 'django.db.utils.DatabaseError'>
['19.0082', '72.81444', '19.0083', '72.81473', '524']	<class 'django.db.utils.DatabaseError'>
['18.98568', '72.82403', '18.98685', '72.82366', '698']	<class 'django.db.utils.DatabaseError'>
['18.98795', '72.82292', '18.98948', '72.82218', '699']	<class 'django.db.utils.DatabaseError'>
['18.99069', '72.82302', '', '72.82305', '699']	<class 'django.db.utils.DatabaseError'>
['18.99323', '72.82008', '18.99362', '72.82007', '700']	<class 'django.db.utils.DatabaseError'>
['18.99352', '72.81995', '18.99376', '72.82008', '700']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '701']	<class 'django.db.utils.DatabaseError'>
['19.01668', '72.82071', '19.01645', '72.82194', '712']	<class 'django.db.utils.DatabaseError'>
['18.99833', '72.82126', '18.99833', '72.82126', '715']	<class 'django.db.utils.DatabaseError'>
['18.99328', '72.82235', '18.99326', '72.82227', '716']	<class 'django.db.utils.DatabaseError'>
['19.01583', '72.82289', '19.01639', '72.82207', '718']	<class 'django.db.utils.DatabaseError'>
['19.00628', '72.82321', '', '', '886']	<class 'django.db.utils.DatabaseError'>
['19.00536', '72.81358', '19.0053', '72.81379', '987']	<class 'django.db.utils.DatabaseError'>
['19.00536', '72.81358', '19.0053', '72.81193', '989']	<class 'django.db.utils.DatabaseError'>
['19.00834', '72.81454', '19.00824', '72.81475', '990']	<class 'django.db.utils.DatabaseError'>
['18.99812', '72.8115', '18.99758', '72.81192', '1075']	<class 'django.db.utils.DatabaseError'>
['19.00205', '72.81175', '19.00218', '72.81197', '1076']	<class 'django.db.utils.DatabaseError'>
['19.0099', '72.81821', '', '', '1077']	<class 'django.db.utils.DatabaseError'>
['18.9948', '72.81385', '18.99483', '72.81385', '1159']	<class 'django.db.utils.DatabaseError'>
['19.00233', '72.81423', '', '', '1664']	<class 'django.db.utils.DatabaseError'>
['19.005', '72.8157', '19.005', '72.8157', '2644']	<class 'django.db.utils.DatabaseError'>
['19.01115', '72.81606', '19.0111', '72.81624', '4064']	<class 'django.db.utils.DatabaseError'>
['19.00536', '72.81358', '19.00766', '72.81193', '']	<type 'exceptions.ValueError'>
['', '', '', '', '128']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '462']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '463']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '595']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '615']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '632']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '656']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '727']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '768']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '774']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '900']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1147']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1161']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1317']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1321']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1431']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1810']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1880']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1896']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1898']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1899']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1900']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1901']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1902']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '1903']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2040']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2149']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2252']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2316']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2476']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2495']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2524']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2575']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2670']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2673']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2674']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2722']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2754']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2907']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2925']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2942']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2943']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '2944']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3274']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3355']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3491']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3492']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3495']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3496']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3497']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3498']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3573']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3602']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3639']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3696']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3766']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3891']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3904']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3905']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3914']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3924']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3925']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3928']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3932']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3933']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3946']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3951']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3952']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3966']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3967']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '3968']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4007']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4023']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4024']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4025']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4037']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4038']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4039']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4040']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4043']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4045']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4055']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4056']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4059']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4060']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4065']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4078']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4084']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4105']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4106']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4107']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4108']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4113']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4141']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4142']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4144']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4154']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4161']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4162']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4163']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4166']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4167']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4168']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4169']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4170']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4171']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4172']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4173']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4174']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4175']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4176']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4177']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4178']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4179']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4180']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4181']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4182']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4183']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4187']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4190']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4192']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4193']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4194']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4195']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4196']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4197']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4198']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4199']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4200']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4201']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4202']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4203']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4204']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4205']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4206']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4207']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4208']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4209']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4210']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4211']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4212']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4213']	<class 'django.db.utils.DatabaseError'>
['', '', '', '', '4214']	<class 'django.db.utils.DatabaseError'>
